{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import re"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option('display.max_columns',None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = pd.read_csv(\"NLP_input.csv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>obj_type</th>\n",
       "      <th>accession</th>\n",
       "      <th>accession_version</th>\n",
       "      <th>title</th>\n",
       "      <th>gene_sort</th>\n",
       "      <th>chr_sort</th>\n",
       "      <th>location_sort</th>\n",
       "      <th>protein_change</th>\n",
       "      <th>fda_recognized_database</th>\n",
       "      <th>scv</th>\n",
       "      <th>rcv</th>\n",
       "      <th>description</th>\n",
       "      <th>last_evaluated</th>\n",
       "      <th>review_status</th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>scv2</th>\n",
       "      <th>#VariationID</th>\n",
       "      <th>ClinicalSignificance</th>\n",
       "      <th>DateLastEvaluated</th>\n",
       "      <th>Description</th>\n",
       "      <th>SubmittedPhenotypeInfo</th>\n",
       "      <th></th>\n",
       "      <th>ReportedPhenotypeInfo</th>\n",
       "      <th>.1</th>\n",
       "      <th>CollectionMethod</th>\n",
       "      <th>OriginCounts</th>\n",
       "      <th>Submitter</th>\n",
       "      <th>SCV</th>\n",
       "      <th>SubmittedGeneSymbol</th>\n",
       "      <th>ExplanationOfInterpretation</th>\n",
       "      <th>PVS1</th>\n",
       "      <th>PVS2</th>\n",
       "      <th>PS2</th>\n",
       "      <th>PS3</th>\n",
       "      <th>PS4</th>\n",
       "      <th>PM1</th>\n",
       "      <th>PM2</th>\n",
       "      <th>PM3</th>\n",
       "      <th>PM4</th>\n",
       "      <th>PM5</th>\n",
       "      <th>PM6</th>\n",
       "      <th>PP1</th>\n",
       "      <th>PP2</th>\n",
       "      <th>PP3</th>\n",
       "      <th>PP4</th>\n",
       "      <th>PP5</th>\n",
       "      <th>BA1</th>\n",
       "      <th>BS1</th>\n",
       "      <th>BS2</th>\n",
       "      <th>BS3</th>\n",
       "      <th>BS4</th>\n",
       "      <th>BP1</th>\n",
       "      <th>BP2</th>\n",
       "      <th>BP3</th>\n",
       "      <th>BP4</th>\n",
       "      <th>BP5</th>\n",
       "      <th>BP6</th>\n",
       "      <th>.2</th>\n",
       "      <th>contributor</th>\n",
       "      <th>FALSE</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000706134</td>\n",
       "      <td>VCV000706134.</td>\n",
       "      <td>NM_000455.5(STK11):c.327T&gt;C</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1218453</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>'SCV001019242', 'SCV001181020'</td>\n",
       "      <td>'RCV000876645', 'RCV001019636'</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-12-2019 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1218452</td>\n",
       "      <td>NM_000455.5(STK11):c.327T&gt;C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=C|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001181020</td>\n",
       "      <td>706134</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>08-Jul-19</td>\n",
       "      <td>In silico models in agreement (benign);Synonym...</td>\n",
       "      <td>MedGen:C0027672</td>\n",
       "      <td>C0027672</td>\n",
       "      <td>Hereditary cancer-predisposing syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:1</td>\n",
       "      <td>Ambry Genetics</td>\n",
       "      <td>SCV001181020</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>deeksha</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000704207</td>\n",
       "      <td>VCV000704207.</td>\n",
       "      <td>NM_000455.5(STK11):c.863-7T&gt;C</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1221942</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCV001016291</td>\n",
       "      <td>RCV000874158</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>27-10-2016 00:00</td>\n",
       "      <td>criteria provided, single submitter</td>\n",
       "      <td>19</td>\n",
       "      <td>1221941</td>\n",
       "      <td>NM_000455.5(STK11):c.863-7T&gt;C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=C|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001016291</td>\n",
       "      <td>704207</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>27-Oct-16</td>\n",
       "      <td>-</td>\n",
       "      <td>MedGen:CN517202</td>\n",
       "      <td>CN517202</td>\n",
       "      <td>not provided</td>\n",
       "      <td>NaN</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>Invitae</td>\n",
       "      <td>SCV001016291</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000702396</td>\n",
       "      <td>VCV000702396.</td>\n",
       "      <td>NM_000455.5(STK11):c.684G&gt;A</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1220667</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>'SCV001013142', 'SCV001187970'</td>\n",
       "      <td>'RCV000871473', 'RCV001025726'</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-12-2019 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1220666</td>\n",
       "      <td>NM_000455.5(STK11):c.684G&gt;A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=A|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001187970</td>\n",
       "      <td>702396</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>12-Jun-18</td>\n",
       "      <td>-</td>\n",
       "      <td>MedGen:C0027672</td>\n",
       "      <td>C0027672</td>\n",
       "      <td>Hereditary cancer-predisposing syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:1</td>\n",
       "      <td>Ambry Genetics</td>\n",
       "      <td>SCV001187970</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>deeksha</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000695193</td>\n",
       "      <td>VCV000695193.</td>\n",
       "      <td>NM_000455.5(STK11):c.1239T&gt;C</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1226584</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCV001000637</td>\n",
       "      <td>RCV000860553</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-12-2019 00:00</td>\n",
       "      <td>criteria provided, single submitter</td>\n",
       "      <td>19</td>\n",
       "      <td>1226583</td>\n",
       "      <td>NM_000455.5(STK11):c.1239T&gt;C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=C|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001000637</td>\n",
       "      <td>695193</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-Dec-19</td>\n",
       "      <td>-</td>\n",
       "      <td>MedGen:C0031269</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>Invitae</td>\n",
       "      <td>SCV001000637</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Deletion</td>\n",
       "      <td>VCV000689700</td>\n",
       "      <td>VCV000689700.</td>\n",
       "      <td>NM_000455.5(STK11):c.418_420del</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1219365</td>\n",
       "      <td>L140del</td>\n",
       "      <td>NaN</td>\n",
       "      <td>'SCV000992655', 'SCV001214728'</td>\n",
       "      <td>'RCV000850457', 'RCV001050610'</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>29-05-2019 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1219365</td>\n",
       "      <td>NM_000455.5(STK11):c.418_420del</td>\n",
       "      <td>GCTG</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=-|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001214728</td>\n",
       "      <td>689700</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>29-May-19</td>\n",
       "      <td>This variant, c.418_420del, results in the del...</td>\n",
       "      <td>MedGen:C0031269</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>Invitae</td>\n",
       "      <td>SCV001214728</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>deeksha</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000007443</td>\n",
       "      <td>VCV000007443.</td>\n",
       "      <td>NM_000455.4(STK11):c.250A&gt;T</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1207163</td>\n",
       "      <td>K84*</td>\n",
       "      <td>NaN</td>\n",
       "      <td>'SCV000028074', 'SCV000889854', 'SCV000541165'...</td>\n",
       "      <td>'RCV000007869', 'RCV000132431', 'RCV000760079'</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>23-01-2020 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1207162</td>\n",
       "      <td>NM_000455.4(STK11):c.250A&gt;T</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=T|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000187525</td>\n",
       "      <td>7443</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>23-Jan-20</td>\n",
       "      <td>The p.K84* pathogenic mutation (also known as ...</td>\n",
       "      <td>MedGen:C0027672</td>\n",
       "      <td>C0027672</td>\n",
       "      <td>Hereditary cancer-predisposing syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:1</td>\n",
       "      <td>Ambry Genetics</td>\n",
       "      <td>SCV000187525</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aatreyee</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000007442</td>\n",
       "      <td>VCV000007442.</td>\n",
       "      <td>NM_000455.4(STK11):c.465-1G&gt;A</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1220372</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCV000028073</td>\n",
       "      <td>RCV000007868</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1220371</td>\n",
       "      <td>NM_000455.4(STK11):c.465-1G&gt;A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=A|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028073</td>\n",
       "      <td>7442</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028073</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>Deletion</td>\n",
       "      <td>VCV000007441</td>\n",
       "      <td>VCV000007441.</td>\n",
       "      <td>NM_000455.4(STK11):c.718_721del</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1220699</td>\n",
       "      <td>S240fs</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCV000028072</td>\n",
       "      <td>RCV000007867</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1220699</td>\n",
       "      <td>NM_000455.4(STK11):c.718_721del</td>\n",
       "      <td>GTCGG</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=-|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028072</td>\n",
       "      <td>7441</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028072</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>Deletion</td>\n",
       "      <td>VCV000007440</td>\n",
       "      <td>VCV000007440.</td>\n",
       "      <td>NM_000455.4(STK11):c.843del</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1221321</td>\n",
       "      <td>L282fs</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCV000028071</td>\n",
       "      <td>RCV000007866</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1221319</td>\n",
       "      <td>NM_000455.4(STK11):c.843del</td>\n",
       "      <td>CG</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=-|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028071</td>\n",
       "      <td>7440</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028071</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000007439</td>\n",
       "      <td>VCV000007439.</td>\n",
       "      <td>NM_000455.4(STK11):c.759C&gt;A</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1221237</td>\n",
       "      <td>Y253*</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCV000028070</td>\n",
       "      <td>RCV000007865</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1221236</td>\n",
       "      <td>NM_000455.4(STK11):c.759C&gt;A</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=A|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028070</td>\n",
       "      <td>7439</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028070</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1202 rows Ã— 67 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       obj_type     accession accession_version  \\\n",
       "0     single nucleotide variant  VCV000706134     VCV000706134.   \n",
       "1     single nucleotide variant  VCV000704207     VCV000704207.   \n",
       "2     single nucleotide variant  VCV000702396     VCV000702396.   \n",
       "3     single nucleotide variant  VCV000695193     VCV000695193.   \n",
       "4                      Deletion  VCV000689700     VCV000689700.   \n",
       "...                         ...           ...               ...   \n",
       "1197  single nucleotide variant  VCV000007443     VCV000007443.   \n",
       "1198  single nucleotide variant  VCV000007442     VCV000007442.   \n",
       "1199                   Deletion  VCV000007441     VCV000007441.   \n",
       "1200                   Deletion  VCV000007440     VCV000007440.   \n",
       "1201  single nucleotide variant  VCV000007439     VCV000007439.   \n",
       "\n",
       "                                title gene_sort  chr_sort  location_sort  \\\n",
       "0         NM_000455.5(STK11):c.327T>C     STK11        19        1218453   \n",
       "1       NM_000455.5(STK11):c.863-7T>C     STK11        19        1221942   \n",
       "2         NM_000455.5(STK11):c.684G>A     STK11        19        1220667   \n",
       "3        NM_000455.5(STK11):c.1239T>C     STK11        19        1226584   \n",
       "4     NM_000455.5(STK11):c.418_420del     STK11        19        1219365   \n",
       "...                               ...       ...       ...            ...   \n",
       "1197      NM_000455.4(STK11):c.250A>T     STK11        19        1207163   \n",
       "1198    NM_000455.4(STK11):c.465-1G>A     STK11        19        1220372   \n",
       "1199  NM_000455.4(STK11):c.718_721del     STK11        19        1220699   \n",
       "1200      NM_000455.4(STK11):c.843del     STK11        19        1221321   \n",
       "1201      NM_000455.4(STK11):c.759C>A     STK11        19        1221237   \n",
       "\n",
       "     protein_change  fda_recognized_database  \\\n",
       "0               NaN                      NaN   \n",
       "1               NaN                      NaN   \n",
       "2               NaN                      NaN   \n",
       "3               NaN                      NaN   \n",
       "4           L140del                      NaN   \n",
       "...             ...                      ...   \n",
       "1197           K84*                      NaN   \n",
       "1198            NaN                      NaN   \n",
       "1199         S240fs                      NaN   \n",
       "1200         L282fs                      NaN   \n",
       "1201          Y253*                      NaN   \n",
       "\n",
       "                                                    scv  \\\n",
       "0                        'SCV001019242', 'SCV001181020'   \n",
       "1                                          SCV001016291   \n",
       "2                        'SCV001013142', 'SCV001187970'   \n",
       "3                                          SCV001000637   \n",
       "4                        'SCV000992655', 'SCV001214728'   \n",
       "...                                                 ...   \n",
       "1197  'SCV000028074', 'SCV000889854', 'SCV000541165'...   \n",
       "1198                                       SCV000028073   \n",
       "1199                                       SCV000028072   \n",
       "1200                                       SCV000028071   \n",
       "1201                                       SCV000028070   \n",
       "\n",
       "                                                 rcv             description  \\\n",
       "0                     'RCV000876645', 'RCV001019636'           Likely benign   \n",
       "1                                       RCV000874158           Likely benign   \n",
       "2                     'RCV000871473', 'RCV001025726'           Likely benign   \n",
       "3                                       RCV000860553           Likely benign   \n",
       "4                     'RCV000850457', 'RCV001050610'  Uncertain significance   \n",
       "...                                              ...                     ...   \n",
       "1197  'RCV000007869', 'RCV000132431', 'RCV000760079'              Pathogenic   \n",
       "1198                                    RCV000007868              Pathogenic   \n",
       "1199                                    RCV000007867              Pathogenic   \n",
       "1200                                    RCV000007866              Pathogenic   \n",
       "1201                                    RCV000007865              Pathogenic   \n",
       "\n",
       "        last_evaluated                                      review_status  \\\n",
       "0     31-12-2019 00:00  criteria provided, multiple submitters, no con...   \n",
       "1     27-10-2016 00:00                criteria provided, single submitter   \n",
       "2     31-12-2019 00:00  criteria provided, multiple submitters, no con...   \n",
       "3     31-12-2019 00:00                criteria provided, single submitter   \n",
       "4     29-05-2019 00:00  criteria provided, multiple submitters, no con...   \n",
       "...                ...                                                ...   \n",
       "1197  23-01-2020 00:00  criteria provided, multiple submitters, no con...   \n",
       "1198  01-01-1998 00:00                     no assertion criteria provided   \n",
       "1199  01-01-1998 00:00                     no assertion criteria provided   \n",
       "1200  01-01-1998 00:00                     no assertion criteria provided   \n",
       "1201  01-01-1998 00:00                     no assertion criteria provided   \n",
       "\n",
       "      #CHROM      POS                               ID    REF ALT QUAL FILTER  \\\n",
       "0         19  1218452      NM_000455.5(STK11):c.327T>C      T   C    .      .   \n",
       "1         19  1221941    NM_000455.5(STK11):c.863-7T>C      T   C    .      .   \n",
       "2         19  1220666      NM_000455.5(STK11):c.684G>A      G   A    .      .   \n",
       "3         19  1226583     NM_000455.5(STK11):c.1239T>C      T   C    .      .   \n",
       "4         19  1219365  NM_000455.5(STK11):c.418_420del   GCTG   G    .      .   \n",
       "...      ...      ...                              ...    ...  ..  ...    ...   \n",
       "1197      19  1207162      NM_000455.4(STK11):c.250A>T      A   T    .      .   \n",
       "1198      19  1220371    NM_000455.4(STK11):c.465-1G>A      G   A    .      .   \n",
       "1199      19  1220699  NM_000455.4(STK11):c.718_721del  GTCGG   G    .      .   \n",
       "1200      19  1221319      NM_000455.4(STK11):c.843del     CG   C    .      .   \n",
       "1201      19  1221236      NM_000455.4(STK11):c.759C>A      C   A    .      .   \n",
       "\n",
       "                                                   INFO          scv2  \\\n",
       "0     CSQ=C|intergenic_variant|MODIFIER|||||||||||||...  SCV001181020   \n",
       "1     CSQ=C|intergenic_variant|MODIFIER|||||||||||||...  SCV001016291   \n",
       "2     CSQ=A|intergenic_variant|MODIFIER|||||||||||||...  SCV001187970   \n",
       "3     CSQ=C|intergenic_variant|MODIFIER|||||||||||||...  SCV001000637   \n",
       "4     CSQ=-|intergenic_variant|MODIFIER|||||||||||||...  SCV001214728   \n",
       "...                                                 ...           ...   \n",
       "1197  CSQ=T|intergenic_variant|MODIFIER|||||||||||||...  SCV000187525   \n",
       "1198  CSQ=A|intergenic_variant|MODIFIER|||||||||||||...  SCV000028073   \n",
       "1199  CSQ=-|intergenic_variant|MODIFIER|||||||||||||...  SCV000028072   \n",
       "1200  CSQ=-|intergenic_variant|MODIFIER|||||||||||||...  SCV000028071   \n",
       "1201  CSQ=A|intergenic_variant|MODIFIER|||||||||||||...  SCV000028070   \n",
       "\n",
       "     #VariationID    ClinicalSignificance DateLastEvaluated  \\\n",
       "0          706134           Likely benign         08-Jul-19   \n",
       "1          704207           Likely benign         27-Oct-16   \n",
       "2          702396           Likely benign         12-Jun-18   \n",
       "3          695193           Likely benign         31-Dec-19   \n",
       "4          689700  Uncertain significance         29-May-19   \n",
       "...           ...                     ...               ...   \n",
       "1197         7443              Pathogenic         23-Jan-20   \n",
       "1198         7442              Pathogenic         01-Jan-98   \n",
       "1199         7441              Pathogenic         01-Jan-98   \n",
       "1200         7440              Pathogenic         01-Jan-98   \n",
       "1201         7439              Pathogenic         01-Jan-98   \n",
       "\n",
       "                                            Description  \\\n",
       "0     In silico models in agreement (benign);Synonym...   \n",
       "1                                                     -   \n",
       "2                                                     -   \n",
       "3                                                     -   \n",
       "4     This variant, c.418_420del, results in the del...   \n",
       "...                                                 ...   \n",
       "1197  The p.K84* pathogenic mutation (also known as ...   \n",
       "1198                                                  -   \n",
       "1199                                                  -   \n",
       "1200                                                  -   \n",
       "1201                                                  -   \n",
       "\n",
       "      SubmittedPhenotypeInfo            \\\n",
       "0            MedGen:C0027672  C0027672   \n",
       "1            MedGen:CN517202  CN517202   \n",
       "2            MedGen:C0027672  C0027672   \n",
       "3            MedGen:C0031269  C0031269   \n",
       "4            MedGen:C0031269  C0031269   \n",
       "...                      ...       ...   \n",
       "1197         MedGen:C0027672  C0027672   \n",
       "1198  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "1199  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "1200  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "1201  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "\n",
       "                        ReportedPhenotypeInfo   .1  CollectionMethod  \\\n",
       "0     Hereditary cancer-predisposing syndrome  NaN  clinical testing   \n",
       "1                                not provided  NaN  clinical testing   \n",
       "2     Hereditary cancer-predisposing syndrome  NaN  clinical testing   \n",
       "3                      Peutz-Jeghers syndrome  NaN  clinical testing   \n",
       "4                      Peutz-Jeghers syndrome  NaN  clinical testing   \n",
       "...                                       ...  ...               ...   \n",
       "1197  Hereditary cancer-predisposing syndrome  NaN  clinical testing   \n",
       "1198                   Peutz-Jeghers syndrome  NaN   literature only   \n",
       "1199                   Peutz-Jeghers syndrome  NaN   literature only   \n",
       "1200                   Peutz-Jeghers syndrome  NaN   literature only   \n",
       "1201                   Peutz-Jeghers syndrome  NaN   literature only   \n",
       "\n",
       "     OriginCounts       Submitter           SCV SubmittedGeneSymbol  \\\n",
       "0      germline:1  Ambry Genetics  SCV001181020               STK11   \n",
       "1     germline:na         Invitae  SCV001016291               STK11   \n",
       "2      germline:1  Ambry Genetics  SCV001187970               STK11   \n",
       "3     germline:na         Invitae  SCV001000637               STK11   \n",
       "4     germline:na         Invitae  SCV001214728               STK11   \n",
       "...           ...             ...           ...                 ...   \n",
       "1197   germline:1  Ambry Genetics  SCV000187525               STK11   \n",
       "1198  germline:na            OMIM  SCV000028073               STK11   \n",
       "1199  germline:na            OMIM  SCV000028072               STK11   \n",
       "1200  germline:na            OMIM  SCV000028071               STK11   \n",
       "1201  germline:na            OMIM  SCV000028070               STK11   \n",
       "\n",
       "     ExplanationOfInterpretation  PVS1  PVS2  PS2  PS3  PS4  PM1  PM2  PM3  \\\n",
       "0                              -   1.0   1.0  NaN  NaN  NaN  1.0  NaN  NaN   \n",
       "1                              -   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "2                              -   NaN   NaN  NaN  NaN  NaN  1.0  NaN  NaN   \n",
       "3                              -   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "4                              -   NaN   NaN  NaN  NaN  NaN  NaN  1.0  NaN   \n",
       "...                          ...   ...   ...  ...  ...  ...  ...  ...  ...   \n",
       "1197                           -   1.0   1.0  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1198                           -   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1199                           -   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1200                           -   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1201                           -   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "\n",
       "      PM4  PM5  PM6  PP1  PP2  PP3  PP4  PP5  BA1  BS1  BS2  BS3  BS4  BP1  \\\n",
       "0     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "2     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "3     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "4     1.0  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "...   ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...   \n",
       "1197  1.0  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1198  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1199  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1200  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1201  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "\n",
       "      BP2  BP3  BP4  BP5  BP6   .2 contributor  FALSE  \n",
       "0     NaN  NaN  1.0  NaN  NaN  NaN     deeksha  False  \n",
       "1     NaN  NaN  NaN  NaN  NaN  NaN         NaN    NaN  \n",
       "2     NaN  NaN  1.0  NaN  NaN  NaN     deeksha  False  \n",
       "3     NaN  NaN  NaN  NaN  NaN  NaN         NaN    NaN  \n",
       "4     NaN  NaN  NaN  NaN  NaN  NaN     deeksha  False  \n",
       "...   ...  ...  ...  ...  ...  ...         ...    ...  \n",
       "1197  NaN  NaN  NaN  NaN  NaN  NaN    Aatreyee  False  \n",
       "1198  NaN  NaN  NaN  NaN  NaN  NaN         NaN    NaN  \n",
       "1199  NaN  NaN  NaN  NaN  NaN  NaN         NaN    NaN  \n",
       "1200  NaN  NaN  NaN  NaN  NaN  NaN         NaN    NaN  \n",
       "1201  NaN  NaN  NaN  NaN  NaN  NaN         NaN    NaN  \n",
       "\n",
       "[1202 rows x 67 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>obj_type</th>\n",
       "      <th>accession</th>\n",
       "      <th>accession_version</th>\n",
       "      <th>title</th>\n",
       "      <th>gene_sort</th>\n",
       "      <th>chr_sort</th>\n",
       "      <th>location_sort</th>\n",
       "      <th>protein_change</th>\n",
       "      <th>fda_recognized_database</th>\n",
       "      <th>scv</th>\n",
       "      <th>rcv</th>\n",
       "      <th>description</th>\n",
       "      <th>last_evaluated</th>\n",
       "      <th>review_status</th>\n",
       "      <th>#CHROM</th>\n",
       "      <th>POS</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>scv2</th>\n",
       "      <th>#VariationID</th>\n",
       "      <th>ClinicalSignificance</th>\n",
       "      <th>DateLastEvaluated</th>\n",
       "      <th>Description</th>\n",
       "      <th>SubmittedPhenotypeInfo</th>\n",
       "      <th></th>\n",
       "      <th>ReportedPhenotypeInfo</th>\n",
       "      <th>.1</th>\n",
       "      <th>CollectionMethod</th>\n",
       "      <th>OriginCounts</th>\n",
       "      <th>Submitter</th>\n",
       "      <th>SCV</th>\n",
       "      <th>SubmittedGeneSymbol</th>\n",
       "      <th>ExplanationOfInterpretation</th>\n",
       "      <th>PVS1</th>\n",
       "      <th>PVS2</th>\n",
       "      <th>PS2</th>\n",
       "      <th>PS3</th>\n",
       "      <th>PS4</th>\n",
       "      <th>PM1</th>\n",
       "      <th>PM2</th>\n",
       "      <th>PM3</th>\n",
       "      <th>PM4</th>\n",
       "      <th>PM5</th>\n",
       "      <th>PM6</th>\n",
       "      <th>PP1</th>\n",
       "      <th>PP2</th>\n",
       "      <th>PP3</th>\n",
       "      <th>PP4</th>\n",
       "      <th>PP5</th>\n",
       "      <th>BA1</th>\n",
       "      <th>BS1</th>\n",
       "      <th>BS2</th>\n",
       "      <th>BS3</th>\n",
       "      <th>BS4</th>\n",
       "      <th>BP1</th>\n",
       "      <th>BP2</th>\n",
       "      <th>BP3</th>\n",
       "      <th>BP4</th>\n",
       "      <th>BP5</th>\n",
       "      <th>BP6</th>\n",
       "      <th>.2</th>\n",
       "      <th>contributor</th>\n",
       "      <th>FALSE</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000706134</td>\n",
       "      <td>VCV000706134.</td>\n",
       "      <td>NM_000455.5(STK11):c.327T&gt;C</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1218453</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>'SCV001019242', 'SCV001181020'</td>\n",
       "      <td>'RCV000876645', 'RCV001019636'</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-12-2019 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1218452</td>\n",
       "      <td>NM_000455.5(STK11):c.327T&gt;C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=C|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001181020</td>\n",
       "      <td>706134</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>08-Jul-19</td>\n",
       "      <td>In silico models in agreement (benign);Synonym...</td>\n",
       "      <td>MedGen:C0027672</td>\n",
       "      <td>C0027672</td>\n",
       "      <td>Hereditary cancer-predisposing syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:1</td>\n",
       "      <td>Ambry Genetics</td>\n",
       "      <td>SCV001181020</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>deeksha</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000704207</td>\n",
       "      <td>VCV000704207.</td>\n",
       "      <td>NM_000455.5(STK11):c.863-7T&gt;C</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1221942</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>SCV001016291</td>\n",
       "      <td>RCV000874158</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>27-10-2016 00:00</td>\n",
       "      <td>criteria provided, single submitter</td>\n",
       "      <td>19</td>\n",
       "      <td>1221941</td>\n",
       "      <td>NM_000455.5(STK11):c.863-7T&gt;C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=C|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001016291</td>\n",
       "      <td>704207</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>27-Oct-16</td>\n",
       "      <td>-</td>\n",
       "      <td>MedGen:CN517202</td>\n",
       "      <td>CN517202</td>\n",
       "      <td>not provided</td>\n",
       "      <td>0</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>Invitae</td>\n",
       "      <td>SCV001016291</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000702396</td>\n",
       "      <td>VCV000702396.</td>\n",
       "      <td>NM_000455.5(STK11):c.684G&gt;A</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1220667</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>'SCV001013142', 'SCV001187970'</td>\n",
       "      <td>'RCV000871473', 'RCV001025726'</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-12-2019 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1220666</td>\n",
       "      <td>NM_000455.5(STK11):c.684G&gt;A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=A|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001187970</td>\n",
       "      <td>702396</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>12-Jun-18</td>\n",
       "      <td>-</td>\n",
       "      <td>MedGen:C0027672</td>\n",
       "      <td>C0027672</td>\n",
       "      <td>Hereditary cancer-predisposing syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:1</td>\n",
       "      <td>Ambry Genetics</td>\n",
       "      <td>SCV001187970</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>deeksha</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000695193</td>\n",
       "      <td>VCV000695193.</td>\n",
       "      <td>NM_000455.5(STK11):c.1239T&gt;C</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1226584</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>SCV001000637</td>\n",
       "      <td>RCV000860553</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-12-2019 00:00</td>\n",
       "      <td>criteria provided, single submitter</td>\n",
       "      <td>19</td>\n",
       "      <td>1226583</td>\n",
       "      <td>NM_000455.5(STK11):c.1239T&gt;C</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=C|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001000637</td>\n",
       "      <td>695193</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>31-Dec-19</td>\n",
       "      <td>-</td>\n",
       "      <td>MedGen:C0031269</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>Invitae</td>\n",
       "      <td>SCV001000637</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Deletion</td>\n",
       "      <td>VCV000689700</td>\n",
       "      <td>VCV000689700.</td>\n",
       "      <td>NM_000455.5(STK11):c.418_420del</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1219365</td>\n",
       "      <td>L140del</td>\n",
       "      <td>0.0</td>\n",
       "      <td>'SCV000992655', 'SCV001214728'</td>\n",
       "      <td>'RCV000850457', 'RCV001050610'</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>29-05-2019 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1219365</td>\n",
       "      <td>NM_000455.5(STK11):c.418_420del</td>\n",
       "      <td>GCTG</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=-|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV001214728</td>\n",
       "      <td>689700</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>29-May-19</td>\n",
       "      <td>This variant, c.418_420del, results in the del...</td>\n",
       "      <td>MedGen:C0031269</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>Invitae</td>\n",
       "      <td>SCV001214728</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>deeksha</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000007443</td>\n",
       "      <td>VCV000007443.</td>\n",
       "      <td>NM_000455.4(STK11):c.250A&gt;T</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1207163</td>\n",
       "      <td>K84*</td>\n",
       "      <td>0.0</td>\n",
       "      <td>'SCV000028074', 'SCV000889854', 'SCV000541165'...</td>\n",
       "      <td>'RCV000007869', 'RCV000132431', 'RCV000760079'</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>23-01-2020 00:00</td>\n",
       "      <td>criteria provided, multiple submitters, no con...</td>\n",
       "      <td>19</td>\n",
       "      <td>1207162</td>\n",
       "      <td>NM_000455.4(STK11):c.250A&gt;T</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=T|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000187525</td>\n",
       "      <td>7443</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>23-Jan-20</td>\n",
       "      <td>The p.K84* pathogenic mutation (also known as ...</td>\n",
       "      <td>MedGen:C0027672</td>\n",
       "      <td>C0027672</td>\n",
       "      <td>Hereditary cancer-predisposing syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>clinical testing</td>\n",
       "      <td>germline:1</td>\n",
       "      <td>Ambry Genetics</td>\n",
       "      <td>SCV000187525</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>Aatreyee</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000007442</td>\n",
       "      <td>VCV000007442.</td>\n",
       "      <td>NM_000455.4(STK11):c.465-1G&gt;A</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1220372</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>SCV000028073</td>\n",
       "      <td>RCV000007868</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1220371</td>\n",
       "      <td>NM_000455.4(STK11):c.465-1G&gt;A</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=A|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028073</td>\n",
       "      <td>7442</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028073</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>Deletion</td>\n",
       "      <td>VCV000007441</td>\n",
       "      <td>VCV000007441.</td>\n",
       "      <td>NM_000455.4(STK11):c.718_721del</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1220699</td>\n",
       "      <td>S240fs</td>\n",
       "      <td>0.0</td>\n",
       "      <td>SCV000028072</td>\n",
       "      <td>RCV000007867</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1220699</td>\n",
       "      <td>NM_000455.4(STK11):c.718_721del</td>\n",
       "      <td>GTCGG</td>\n",
       "      <td>G</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=-|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028072</td>\n",
       "      <td>7441</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028072</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>Deletion</td>\n",
       "      <td>VCV000007440</td>\n",
       "      <td>VCV000007440.</td>\n",
       "      <td>NM_000455.4(STK11):c.843del</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1221321</td>\n",
       "      <td>L282fs</td>\n",
       "      <td>0.0</td>\n",
       "      <td>SCV000028071</td>\n",
       "      <td>RCV000007866</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1221319</td>\n",
       "      <td>NM_000455.4(STK11):c.843del</td>\n",
       "      <td>CG</td>\n",
       "      <td>C</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=-|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028071</td>\n",
       "      <td>7440</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028071</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>single nucleotide variant</td>\n",
       "      <td>VCV000007439</td>\n",
       "      <td>VCV000007439.</td>\n",
       "      <td>NM_000455.4(STK11):c.759C&gt;A</td>\n",
       "      <td>STK11</td>\n",
       "      <td>19</td>\n",
       "      <td>1221237</td>\n",
       "      <td>Y253*</td>\n",
       "      <td>0.0</td>\n",
       "      <td>SCV000028070</td>\n",
       "      <td>RCV000007865</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-01-1998 00:00</td>\n",
       "      <td>no assertion criteria provided</td>\n",
       "      <td>19</td>\n",
       "      <td>1221236</td>\n",
       "      <td>NM_000455.4(STK11):c.759C&gt;A</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>CSQ=A|intergenic_variant|MODIFIER|||||||||||||...</td>\n",
       "      <td>SCV000028070</td>\n",
       "      <td>7439</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>01-Jan-98</td>\n",
       "      <td>-</td>\n",
       "      <td>PEUTZ-JEGHERS SYNDROME</td>\n",
       "      <td>C0031269</td>\n",
       "      <td>Peutz-Jeghers syndrome</td>\n",
       "      <td>0</td>\n",
       "      <td>literature only</td>\n",
       "      <td>germline:na</td>\n",
       "      <td>OMIM</td>\n",
       "      <td>SCV000028070</td>\n",
       "      <td>STK11</td>\n",
       "      <td>-</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1202 rows Ã— 67 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       obj_type     accession accession_version  \\\n",
       "0     single nucleotide variant  VCV000706134     VCV000706134.   \n",
       "1     single nucleotide variant  VCV000704207     VCV000704207.   \n",
       "2     single nucleotide variant  VCV000702396     VCV000702396.   \n",
       "3     single nucleotide variant  VCV000695193     VCV000695193.   \n",
       "4                      Deletion  VCV000689700     VCV000689700.   \n",
       "...                         ...           ...               ...   \n",
       "1197  single nucleotide variant  VCV000007443     VCV000007443.   \n",
       "1198  single nucleotide variant  VCV000007442     VCV000007442.   \n",
       "1199                   Deletion  VCV000007441     VCV000007441.   \n",
       "1200                   Deletion  VCV000007440     VCV000007440.   \n",
       "1201  single nucleotide variant  VCV000007439     VCV000007439.   \n",
       "\n",
       "                                title gene_sort  chr_sort  location_sort  \\\n",
       "0         NM_000455.5(STK11):c.327T>C     STK11        19        1218453   \n",
       "1       NM_000455.5(STK11):c.863-7T>C     STK11        19        1221942   \n",
       "2         NM_000455.5(STK11):c.684G>A     STK11        19        1220667   \n",
       "3        NM_000455.5(STK11):c.1239T>C     STK11        19        1226584   \n",
       "4     NM_000455.5(STK11):c.418_420del     STK11        19        1219365   \n",
       "...                               ...       ...       ...            ...   \n",
       "1197      NM_000455.4(STK11):c.250A>T     STK11        19        1207163   \n",
       "1198    NM_000455.4(STK11):c.465-1G>A     STK11        19        1220372   \n",
       "1199  NM_000455.4(STK11):c.718_721del     STK11        19        1220699   \n",
       "1200      NM_000455.4(STK11):c.843del     STK11        19        1221321   \n",
       "1201      NM_000455.4(STK11):c.759C>A     STK11        19        1221237   \n",
       "\n",
       "     protein_change  fda_recognized_database  \\\n",
       "0                 0                      0.0   \n",
       "1                 0                      0.0   \n",
       "2                 0                      0.0   \n",
       "3                 0                      0.0   \n",
       "4           L140del                      0.0   \n",
       "...             ...                      ...   \n",
       "1197           K84*                      0.0   \n",
       "1198              0                      0.0   \n",
       "1199         S240fs                      0.0   \n",
       "1200         L282fs                      0.0   \n",
       "1201          Y253*                      0.0   \n",
       "\n",
       "                                                    scv  \\\n",
       "0                        'SCV001019242', 'SCV001181020'   \n",
       "1                                          SCV001016291   \n",
       "2                        'SCV001013142', 'SCV001187970'   \n",
       "3                                          SCV001000637   \n",
       "4                        'SCV000992655', 'SCV001214728'   \n",
       "...                                                 ...   \n",
       "1197  'SCV000028074', 'SCV000889854', 'SCV000541165'...   \n",
       "1198                                       SCV000028073   \n",
       "1199                                       SCV000028072   \n",
       "1200                                       SCV000028071   \n",
       "1201                                       SCV000028070   \n",
       "\n",
       "                                                 rcv             description  \\\n",
       "0                     'RCV000876645', 'RCV001019636'           Likely benign   \n",
       "1                                       RCV000874158           Likely benign   \n",
       "2                     'RCV000871473', 'RCV001025726'           Likely benign   \n",
       "3                                       RCV000860553           Likely benign   \n",
       "4                     'RCV000850457', 'RCV001050610'  Uncertain significance   \n",
       "...                                              ...                     ...   \n",
       "1197  'RCV000007869', 'RCV000132431', 'RCV000760079'              Pathogenic   \n",
       "1198                                    RCV000007868              Pathogenic   \n",
       "1199                                    RCV000007867              Pathogenic   \n",
       "1200                                    RCV000007866              Pathogenic   \n",
       "1201                                    RCV000007865              Pathogenic   \n",
       "\n",
       "        last_evaluated                                      review_status  \\\n",
       "0     31-12-2019 00:00  criteria provided, multiple submitters, no con...   \n",
       "1     27-10-2016 00:00                criteria provided, single submitter   \n",
       "2     31-12-2019 00:00  criteria provided, multiple submitters, no con...   \n",
       "3     31-12-2019 00:00                criteria provided, single submitter   \n",
       "4     29-05-2019 00:00  criteria provided, multiple submitters, no con...   \n",
       "...                ...                                                ...   \n",
       "1197  23-01-2020 00:00  criteria provided, multiple submitters, no con...   \n",
       "1198  01-01-1998 00:00                     no assertion criteria provided   \n",
       "1199  01-01-1998 00:00                     no assertion criteria provided   \n",
       "1200  01-01-1998 00:00                     no assertion criteria provided   \n",
       "1201  01-01-1998 00:00                     no assertion criteria provided   \n",
       "\n",
       "      #CHROM      POS                               ID    REF ALT QUAL FILTER  \\\n",
       "0         19  1218452      NM_000455.5(STK11):c.327T>C      T   C    .      .   \n",
       "1         19  1221941    NM_000455.5(STK11):c.863-7T>C      T   C    .      .   \n",
       "2         19  1220666      NM_000455.5(STK11):c.684G>A      G   A    .      .   \n",
       "3         19  1226583     NM_000455.5(STK11):c.1239T>C      T   C    .      .   \n",
       "4         19  1219365  NM_000455.5(STK11):c.418_420del   GCTG   G    .      .   \n",
       "...      ...      ...                              ...    ...  ..  ...    ...   \n",
       "1197      19  1207162      NM_000455.4(STK11):c.250A>T      A   T    .      .   \n",
       "1198      19  1220371    NM_000455.4(STK11):c.465-1G>A      G   A    .      .   \n",
       "1199      19  1220699  NM_000455.4(STK11):c.718_721del  GTCGG   G    .      .   \n",
       "1200      19  1221319      NM_000455.4(STK11):c.843del     CG   C    .      .   \n",
       "1201      19  1221236      NM_000455.4(STK11):c.759C>A      C   A    .      .   \n",
       "\n",
       "                                                   INFO          scv2  \\\n",
       "0     CSQ=C|intergenic_variant|MODIFIER|||||||||||||...  SCV001181020   \n",
       "1     CSQ=C|intergenic_variant|MODIFIER|||||||||||||...  SCV001016291   \n",
       "2     CSQ=A|intergenic_variant|MODIFIER|||||||||||||...  SCV001187970   \n",
       "3     CSQ=C|intergenic_variant|MODIFIER|||||||||||||...  SCV001000637   \n",
       "4     CSQ=-|intergenic_variant|MODIFIER|||||||||||||...  SCV001214728   \n",
       "...                                                 ...           ...   \n",
       "1197  CSQ=T|intergenic_variant|MODIFIER|||||||||||||...  SCV000187525   \n",
       "1198  CSQ=A|intergenic_variant|MODIFIER|||||||||||||...  SCV000028073   \n",
       "1199  CSQ=-|intergenic_variant|MODIFIER|||||||||||||...  SCV000028072   \n",
       "1200  CSQ=-|intergenic_variant|MODIFIER|||||||||||||...  SCV000028071   \n",
       "1201  CSQ=A|intergenic_variant|MODIFIER|||||||||||||...  SCV000028070   \n",
       "\n",
       "     #VariationID    ClinicalSignificance DateLastEvaluated  \\\n",
       "0          706134           Likely benign         08-Jul-19   \n",
       "1          704207           Likely benign         27-Oct-16   \n",
       "2          702396           Likely benign         12-Jun-18   \n",
       "3          695193           Likely benign         31-Dec-19   \n",
       "4          689700  Uncertain significance         29-May-19   \n",
       "...           ...                     ...               ...   \n",
       "1197         7443              Pathogenic         23-Jan-20   \n",
       "1198         7442              Pathogenic         01-Jan-98   \n",
       "1199         7441              Pathogenic         01-Jan-98   \n",
       "1200         7440              Pathogenic         01-Jan-98   \n",
       "1201         7439              Pathogenic         01-Jan-98   \n",
       "\n",
       "                                            Description  \\\n",
       "0     In silico models in agreement (benign);Synonym...   \n",
       "1                                                     -   \n",
       "2                                                     -   \n",
       "3                                                     -   \n",
       "4     This variant, c.418_420del, results in the del...   \n",
       "...                                                 ...   \n",
       "1197  The p.K84* pathogenic mutation (also known as ...   \n",
       "1198                                                  -   \n",
       "1199                                                  -   \n",
       "1200                                                  -   \n",
       "1201                                                  -   \n",
       "\n",
       "      SubmittedPhenotypeInfo            \\\n",
       "0            MedGen:C0027672  C0027672   \n",
       "1            MedGen:CN517202  CN517202   \n",
       "2            MedGen:C0027672  C0027672   \n",
       "3            MedGen:C0031269  C0031269   \n",
       "4            MedGen:C0031269  C0031269   \n",
       "...                      ...       ...   \n",
       "1197         MedGen:C0027672  C0027672   \n",
       "1198  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "1199  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "1200  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "1201  PEUTZ-JEGHERS SYNDROME  C0031269   \n",
       "\n",
       "                        ReportedPhenotypeInfo  .1  CollectionMethod  \\\n",
       "0     Hereditary cancer-predisposing syndrome   0  clinical testing   \n",
       "1                                not provided   0  clinical testing   \n",
       "2     Hereditary cancer-predisposing syndrome   0  clinical testing   \n",
       "3                      Peutz-Jeghers syndrome   0  clinical testing   \n",
       "4                      Peutz-Jeghers syndrome   0  clinical testing   \n",
       "...                                       ...  ..               ...   \n",
       "1197  Hereditary cancer-predisposing syndrome   0  clinical testing   \n",
       "1198                   Peutz-Jeghers syndrome   0   literature only   \n",
       "1199                   Peutz-Jeghers syndrome   0   literature only   \n",
       "1200                   Peutz-Jeghers syndrome   0   literature only   \n",
       "1201                   Peutz-Jeghers syndrome   0   literature only   \n",
       "\n",
       "     OriginCounts       Submitter           SCV SubmittedGeneSymbol  \\\n",
       "0      germline:1  Ambry Genetics  SCV001181020               STK11   \n",
       "1     germline:na         Invitae  SCV001016291               STK11   \n",
       "2      germline:1  Ambry Genetics  SCV001187970               STK11   \n",
       "3     germline:na         Invitae  SCV001000637               STK11   \n",
       "4     germline:na         Invitae  SCV001214728               STK11   \n",
       "...           ...             ...           ...                 ...   \n",
       "1197   germline:1  Ambry Genetics  SCV000187525               STK11   \n",
       "1198  germline:na            OMIM  SCV000028073               STK11   \n",
       "1199  germline:na            OMIM  SCV000028072               STK11   \n",
       "1200  germline:na            OMIM  SCV000028071               STK11   \n",
       "1201  germline:na            OMIM  SCV000028070               STK11   \n",
       "\n",
       "     ExplanationOfInterpretation  PVS1  PVS2  PS2  PS3  PS4  PM1  PM2  PM3  \\\n",
       "0                              -   1.0   1.0  0.0  0.0  0.0  1.0  0.0  0.0   \n",
       "1                              -   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "2                              -   0.0   0.0  0.0  0.0  0.0  1.0  0.0  0.0   \n",
       "3                              -   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "4                              -   0.0   0.0  0.0  0.0  0.0  0.0  1.0  0.0   \n",
       "...                          ...   ...   ...  ...  ...  ...  ...  ...  ...   \n",
       "1197                           -   1.0   1.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1198                           -   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1199                           -   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1200                           -   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1201                           -   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "\n",
       "      PM4  PM5  PM6  PP1  PP2  PP3  PP4  PP5  BA1  BS1  BS2  BS3  BS4  BP1  \\\n",
       "0     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "2     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "3     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "4     1.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "...   ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...   \n",
       "1197  1.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1198  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1199  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1200  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1201  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "\n",
       "      BP2  BP3  BP4  BP5  BP6   .2 contributor  FALSE  \n",
       "0     0.0  0.0  1.0  0.0  0.0  0.0     deeksha  False  \n",
       "1     0.0  0.0  0.0  0.0  0.0  0.0           0      0  \n",
       "2     0.0  0.0  1.0  0.0  0.0  0.0     deeksha  False  \n",
       "3     0.0  0.0  0.0  0.0  0.0  0.0           0      0  \n",
       "4     0.0  0.0  0.0  0.0  0.0  0.0     deeksha  False  \n",
       "...   ...  ...  ...  ...  ...  ...         ...    ...  \n",
       "1197  0.0  0.0  0.0  0.0  0.0  0.0    Aatreyee  False  \n",
       "1198  0.0  0.0  0.0  0.0  0.0  0.0           0      0  \n",
       "1199  0.0  0.0  0.0  0.0  0.0  0.0           0      0  \n",
       "1200  0.0  0.0  0.0  0.0  0.0  0.0           0      0  \n",
       "1201  0.0  0.0  0.0  0.0  0.0  0.0           0      0  \n",
       "\n",
       "[1202 rows x 67 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.fillna(value =0 )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "##dataset.loc['PVS1','ps1','PS2','PS3','PS4','PM1','PM2','PM3','PM4','PM5','PM6','PP1','PP2','PP3','PP4','PP5','BA1','BS1','BS2','BS3','BS4','BP1','BP2','BP3','BP4','BP5','BP6']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "df =dataset.iloc[: ,37:64]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PVS1</th>\n",
       "      <th>PVS2</th>\n",
       "      <th>PS2</th>\n",
       "      <th>PS3</th>\n",
       "      <th>PS4</th>\n",
       "      <th>PM1</th>\n",
       "      <th>PM2</th>\n",
       "      <th>PM3</th>\n",
       "      <th>PM4</th>\n",
       "      <th>PM5</th>\n",
       "      <th>PM6</th>\n",
       "      <th>PP1</th>\n",
       "      <th>PP2</th>\n",
       "      <th>PP3</th>\n",
       "      <th>PP4</th>\n",
       "      <th>PP5</th>\n",
       "      <th>BA1</th>\n",
       "      <th>BS1</th>\n",
       "      <th>BS2</th>\n",
       "      <th>BS3</th>\n",
       "      <th>BS4</th>\n",
       "      <th>BP1</th>\n",
       "      <th>BP2</th>\n",
       "      <th>BP3</th>\n",
       "      <th>BP4</th>\n",
       "      <th>BP5</th>\n",
       "      <th>BP6</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1202 rows Ã— 27 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      PVS1  PVS2  PS2  PS3  PS4  PM1  PM2  PM3  PM4  PM5  PM6  PP1  PP2  PP3  \\\n",
       "0      1.0   1.0  NaN  NaN  NaN  1.0  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1      NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "2      NaN   NaN  NaN  NaN  NaN  1.0  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "3      NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "4      NaN   NaN  NaN  NaN  NaN  NaN  1.0  NaN  1.0  NaN  NaN  NaN  NaN  NaN   \n",
       "...    ...   ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...   \n",
       "1197   1.0   1.0  NaN  NaN  NaN  NaN  NaN  NaN  1.0  NaN  NaN  NaN  NaN  NaN   \n",
       "1198   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1199   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1200   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "1201   NaN   NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN   \n",
       "\n",
       "      PP4  PP5  BA1  BS1  BS2  BS3  BS4  BP1  BP2  BP3  BP4  BP5  BP6  \n",
       "0     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  1.0  NaN  NaN  \n",
       "1     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "2     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  1.0  NaN  NaN  \n",
       "3     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "4     NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "...   ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  \n",
       "1197  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "1198  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "1199  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "1200  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "1201  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  NaN  \n",
       "\n",
       "[1202 rows x 27 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "df =df.fillna(value = 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PVS1</th>\n",
       "      <th>PVS2</th>\n",
       "      <th>PS2</th>\n",
       "      <th>PS3</th>\n",
       "      <th>PS4</th>\n",
       "      <th>PM1</th>\n",
       "      <th>PM2</th>\n",
       "      <th>PM3</th>\n",
       "      <th>PM4</th>\n",
       "      <th>PM5</th>\n",
       "      <th>PM6</th>\n",
       "      <th>PP1</th>\n",
       "      <th>PP2</th>\n",
       "      <th>PP3</th>\n",
       "      <th>PP4</th>\n",
       "      <th>PP5</th>\n",
       "      <th>BA1</th>\n",
       "      <th>BS1</th>\n",
       "      <th>BS2</th>\n",
       "      <th>BS3</th>\n",
       "      <th>BS4</th>\n",
       "      <th>BP1</th>\n",
       "      <th>BP2</th>\n",
       "      <th>BP3</th>\n",
       "      <th>BP4</th>\n",
       "      <th>BP5</th>\n",
       "      <th>BP6</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1202 rows Ã— 27 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      PVS1  PVS2  PS2  PS3  PS4  PM1  PM2  PM3  PM4  PM5  PM6  PP1  PP2  PP3  \\\n",
       "0      1.0   1.0  0.0  0.0  0.0  1.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1      0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "2      0.0   0.0  0.0  0.0  0.0  1.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "3      0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "4      0.0   0.0  0.0  0.0  0.0  0.0  1.0  0.0  1.0  0.0  0.0  0.0  0.0  0.0   \n",
       "...    ...   ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...   \n",
       "1197   1.0   1.0  0.0  0.0  0.0  0.0  0.0  0.0  1.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1198   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1199   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1200   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "1201   0.0   0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0   \n",
       "\n",
       "      PP4  PP5  BA1  BS1  BS2  BS3  BS4  BP1  BP2  BP3  BP4  BP5  BP6  \n",
       "0     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  1.0  0.0  0.0  \n",
       "1     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "2     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  1.0  0.0  0.0  \n",
       "3     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "4     0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "...   ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  ...  \n",
       "1197  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "1198  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "1199  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "1200  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "1201  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  0.0  \n",
       "\n",
       "[1202 rows x 27 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "Y = df.iloc[:,:].values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[1., 1., 0., ..., 1., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 1., 0., 0.],\n",
       "       ...,\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.]])"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Y"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0       In silico models in agreement (benign);Synonym...\n",
       "1                                                       -\n",
       "2                                                       -\n",
       "3                                                       -\n",
       "4       This variant, c.418_420del, results in the del...\n",
       "                              ...                        \n",
       "1197    The p.K84* pathogenic mutation (also known as ...\n",
       "1198                                                    -\n",
       "1199                                                    -\n",
       "1200                                                    -\n",
       "1201                                                    -\n",
       "Name: Description, Length: 1202, dtype: object"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset['Description']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 1202 entries, 0 to 1201\n",
      "Data columns (total 67 columns):\n",
      " #   Column                       Non-Null Count  Dtype  \n",
      "---  ------                       --------------  -----  \n",
      " 0   obj_type                     1202 non-null   object \n",
      " 1   accession                    1202 non-null   object \n",
      " 2   accession_version            1202 non-null   object \n",
      " 3   title                        1202 non-null   object \n",
      " 4   gene_sort                    1202 non-null   object \n",
      " 5   chr_sort                     1202 non-null   int64  \n",
      " 6   location_sort                1202 non-null   int64  \n",
      " 7   protein_change               571 non-null    object \n",
      " 8   fda_recognized_database      0 non-null      float64\n",
      " 9   scv                          1202 non-null   object \n",
      " 10  rcv                          1202 non-null   object \n",
      " 11  description                  1202 non-null   object \n",
      " 12  last_evaluated               1202 non-null   object \n",
      " 13  review_status                1202 non-null   object \n",
      " 14  #CHROM                       1202 non-null   int64  \n",
      " 15  POS                          1202 non-null   int64  \n",
      " 16  ID                           1202 non-null   object \n",
      " 17  REF                          1202 non-null   object \n",
      " 18  ALT                          1202 non-null   object \n",
      " 19  QUAL                         1202 non-null   object \n",
      " 20  FILTER                       1202 non-null   object \n",
      " 21  INFO                         1202 non-null   object \n",
      " 22  scv2                         1202 non-null   object \n",
      " 23  #VariationID                 1202 non-null   object \n",
      " 24  ClinicalSignificance         1202 non-null   object \n",
      " 25  DateLastEvaluated            1202 non-null   object \n",
      " 26  Description                  1202 non-null   object \n",
      " 27  SubmittedPhenotypeInfo       1202 non-null   object \n",
      " 28                               1202 non-null   object \n",
      " 29  ReportedPhenotypeInfo        1202 non-null   object \n",
      " 30   .1                          1 non-null      object \n",
      " 31  CollectionMethod             1202 non-null   object \n",
      " 32  OriginCounts                 1202 non-null   object \n",
      " 33  Submitter                    1202 non-null   object \n",
      " 34  SCV                          1202 non-null   object \n",
      " 35  SubmittedGeneSymbol          1202 non-null   object \n",
      " 36  ExplanationOfInterpretation  1202 non-null   object \n",
      " 37  PVS1                         85 non-null     float64\n",
      " 38  PVS2                         85 non-null     float64\n",
      " 39  PS2                          2 non-null      float64\n",
      " 40  PS3                          1 non-null      float64\n",
      " 41  PS4                          43 non-null     float64\n",
      " 42  PM1                          9 non-null      float64\n",
      " 43  PM2                          237 non-null    float64\n",
      " 44  PM3                          0 non-null      float64\n",
      " 45  PM4                          38 non-null     float64\n",
      " 46  PM5                          63 non-null     float64\n",
      " 47  PM6                          2 non-null      float64\n",
      " 48  PP1                          0 non-null      float64\n",
      " 49  PP2                          3 non-null      float64\n",
      " 50  PP3                          211 non-null    float64\n",
      " 51  PP4                          1 non-null      float64\n",
      " 52  PP5                          75 non-null     float64\n",
      " 53  BA1                          3 non-null      float64\n",
      " 54  BS1                          177 non-null    float64\n",
      " 55  BS2                          18 non-null     float64\n",
      " 56  BS3                          1 non-null      float64\n",
      " 57  BS4                          87 non-null     float64\n",
      " 58  BP1                          1 non-null      float64\n",
      " 59  BP2                          0 non-null      float64\n",
      " 60  BP3                          0 non-null      float64\n",
      " 61  BP4                          325 non-null    float64\n",
      " 62  BP5                          1 non-null      float64\n",
      " 63  BP6                          83 non-null     float64\n",
      " 64   .2                          0 non-null      float64\n",
      " 65  contributor                  722 non-null    object \n",
      " 66  FALSE                        800 non-null    object \n",
      "dtypes: float64(29), int64(4), object(34)\n",
      "memory usage: 629.3+ KB\n"
     ]
    }
   ],
   "source": [
    "dataset.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_data =dataset['Description']\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0       In silico models in agreement (benign);Synonym...\n",
       "1                                                       -\n",
       "2                                                       -\n",
       "3                                                       -\n",
       "4       This variant, c.418_420del, results in the del...\n",
       "                              ...                        \n",
       "1197    The p.K84* pathogenic mutation (also known as ...\n",
       "1198                                                    -\n",
       "1199                                                    -\n",
       "1200                                                    -\n",
       "1201                                                    -\n",
       "Name: Description, Length: 1202, dtype: object"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "def decontraction(phrase):\n",
    "    \n",
    "    phrase = re.sub(r\"won\\'t\", \"will not\", phrase)\n",
    "    phrase = re.sub(r\"can\\'t\", \"can not\", phrase)\n",
    "    phrase = re.sub(r\"n\\'t\", \" not\", phrase)\n",
    "    phrase = re.sub(r\"\\'re\", \" are\", phrase)\n",
    "    phrase = re.sub(r\"\\'s\", \" is\", phrase)\n",
    "    phrase = re.sub(r\"\\'d\", \" would\", phrase)\n",
    "    phrase = re.sub(r\"\\'ll\", \" will\", phrase)\n",
    "    phrase = re.sub(r\"\\'t\", \" not\", phrase)\n",
    "    phrase =phrase.lower()\n",
    "    phrase = re.sub(r\"\\'ve\", \" have\", phrase)\n",
    "    phrase = re.sub(r\"\\'m\", \" am\", phrase)\n",
    "    phrase = re.sub('\\[.*?\\]', ' ', phrase) \n",
    "    phrase = re.sub('https?://\\S+|www\\.\\S+', ' ', phrase)\n",
    "    phrase = re.sub('<.*?>+', ' ', phrase)\n",
    "    phrase = re.sub('\\n', ' ', phrase)\n",
    "    phrase = re.sub('\\w*\\d\\w*', ' ', phrase)\n",
    "    return phrase\n",
    "\n",
    "dataset.Description = [decontraction(word) for word in dataset.Description]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0       in silico models in agreement (benign);synonym...\n",
       "1                                                       -\n",
       "2                                                       -\n",
       "3                                                       -\n",
       "4       this variant, c. , results in the deletion of ...\n",
       "                              ...                        \n",
       "1197    the p. * pathogenic mutation (also known as c....\n",
       "1198                                                    -\n",
       "1199                                                    -\n",
       "1200                                                    -\n",
       "1201                                                    -\n",
       "Name: Description, Length: 1202, dtype: object"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.Description"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "import nltk\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.stem import WordNetLemmatizer \n",
    "from nltk.stem import PorterStemmer\n",
    "lemmatizer = WordNetLemmatizer()\n",
    "ps = PorterStemmer()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\dell\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n",
      "[nltk_data] Downloading package punkt to\n",
      "[nltk_data]     C:\\Users\\dell\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package punkt is already up-to-date!\n",
      "[nltk_data] Downloading package wordnet to\n",
      "[nltk_data]     C:\\Users\\dell\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package wordnet is already up-to-date!\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "nltk.download('stopwords')\n",
    "nltk.download('punkt')\n",
    "nltk.download('wordnet')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['silico model agreement benign synonymous alteration insufficient evidence classify benign', '', '', '', 'variant c result deletion amino acid protein p otherwise preserve integrity reading frame variant not present population database exac frequency variant not reported literature individual related condition experimental study prediction algorithm not available not evaluated variant functional significance affected amino acid currently unknown summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', '', 'sequence change replaces asparagine histidine codon protein p asparagine residue highly conserved small physicochemical difference asparagine histidine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related condition loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces glycine cysteine codon protein p glycine residue weakly conserved large physicochemical difference glycine cysteine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant observed individual affected peutz jeghers syndrome pjs pmid loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces serine phenylalanine codon protein p serine residue highly conserved large physicochemical difference serine phenylalanine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces threonine isoleucine codon protein p threonine residue moderately conserved moderate physicochemical difference threonine isoleucine variant not present population database exac frequency variant not reported literature individual related condition algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene methionine codon replaced isoleucine amino acid highly similar property change occurs first base pair coding exon amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related condition loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces glutamine lysine codon protein p glutamine residue moderately conserved small physicochemical difference glutamine lysine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen not available align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change fall intron gene not directly change encoded amino acid sequence protein affect nucleotide within consensus splice site intron variant not present population database exac frequency variant not reported literature individual related disease nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant not likely affect rna splicing prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces alanine aspartic acid codon protein p alanine residue moderately conserved moderate physicochemical difference alanine aspartic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces cysteine tyrosine codon protein p cysteine residue moderately conserved large physicochemical difference cysteine tyrosine frequency data variant population database considered unreliable metric indicate insufficient coverage position exac database variant not reported literature individual related condition algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant not present population database exac frequency variant observed individual affected colorectal cancer pmid algorithm developed predict effect sequence change rna splicing suggest variant not likely affect rna splicing prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect donor splice site intron gene expected disrupt rna splicing likely result absent disrupted protein product variant not present population database exac frequency variant observed individual affected peutz jeghers syndrome pmid donor acceptor splice site variant typically lead loss protein function pmid loss function variant known pathogenic pmid summary currently available evidence indicates variant pathogenic additional data needed prove conclusively therefore variant classified likely pathogenic', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'variant c result deletion amino acid protein p otherwise preserve integrity reading frame variant not present population database exac frequency variant not reported literature individual related disease experimental study prediction algorithm not available variant functional significance affected amino acid currently unknown summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect acceptor splice site intron gene expected disrupt rna splicing likely result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease donor acceptor splice site variant typically lead loss protein function pmid loss function variant known pathogenic pmid summary currently available evidence indicates variant pathogenic additional data needed prove conclusively therefore variant classified likely pathogenic', 'sequence change replaces serine threonine codon protein p serine residue moderately conserved small physicochemical difference serine threonine frequency data variant population database considered unreliable metric indicate insufficient coverage position exac database variant not reported literature individual related condition algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces asparagine lysine codon protein p asparagine residue moderately conserved moderate physicochemical difference asparagine lysine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glycine alanine codon protein p glycine residue highly conserved small physicochemical difference glycine alanine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces proline threonine codon protein p proline residue moderately conserved small physicochemical difference proline threonine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces lysine arginine codon protein p lysine residue highly conserved small physicochemical difference lysine arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces lysine asparagine codon protein p lysine residue highly conserved moderate physicochemical difference lysine asparagine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glutamic acid glutamine codon protein p glutamic acid residue weakly conserved small physicochemical difference glutamic acid glutamine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine alanine codon protein p valine residue highly conserved small physicochemical difference valine alanine variant not present population database exac frequency variant not reported literature individual related condition algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant present population database exac variant not reported literature individual related disease algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces proline serine codon protein p proline residue highly conserved moderate physicochemical difference proline serine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced arginine amino acid dissimilar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces methionine isoleucine codon protein p methionine residue highly conserved small physicochemical difference methionine isoleucine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property though alteration not reported literature germline alteration one study detected p sporadic colorectal cancer sample functional study demonstrated variant not affect kinase function launonen v et al cancer re feb amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position methionine codon replaced valine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces arginine leucine codon protein p arginine residue weakly conserved moderate physicochemical difference arginine leucine variant not present population database exac frequency variant not reported literature individual related condition algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class leucine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces threonine proline codon protein p threonine residue highly conserved small physicochemical difference threonine proline variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces arginine leucine codon protein p arginine residue moderately conserved moderate physicochemical difference arginine leucine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced serine amino acid similar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glutamine glutamic acid codon protein p glutamine residue highly conserved small physicochemical difference glutamine glutamic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glycine arginine codon protein p glycine residue highly conserved moderate physicochemical difference glycine arginine variant also fall last nucleotide exon coding sequence part consensus splice site exon variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant may disrupt consensus splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces arginine histidine codon protein p arginine residue highly conserved small physicochemical difference arginine histidine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces proline arginine codon protein p proline residue highly conserved moderate physicochemical difference proline arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine methionine codon protein p valine residue moderately conserved small physicochemical difference valine methionine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class methionine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces lysine asparagine codon protein p lysine residue highly conserved moderate physicochemical difference lysine asparagine frequency data variant population database considered unreliable metric indicate insufficient coverage position exac database variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces serine asparagine codon protein p serine residue highly conserved small physicochemical difference serine asparagine variant not present population database exac frequency variant not reported literature individual related condition algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'variant summary c alters non conserved nucleotide located close canonical splice site therefore could affect mrna splicing leading significantly altered protein sequence several computational tool predict significant impact normal splicing tool predict variant disrupt canonical splice site create possibly utilize cryptic acceptor site however prediction yet confirmed functional study available data variant occurrence general population insufficient allow conclusion variant significance knowledge occurrence c individual affected peutz jeghers syndrome experimental evidence demonstrating impact protein function reported clinical diagnostic laboratory submitted clinical significance assessment variant clinvar based evidence outlined variant classified uncertain significance', '', '', '', '', '', '', 'p variant also known c located coding exon gene result g substitution nucleotide position aspartic acid codon replaced asparagine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', '', '', '', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property variant reported study spanish breast ovarian cancer family without mutation bonache et al j cancer re clin oncol dec amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces leucine glutamine codon protein p leucine residue highly conserved moderate physicochemical difference leucine glutamine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position alanine codon replaced threonine amino acid similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 'p variant also known c g located coding exon gene result g substitution nucleotide position glutamine codon replaced arginine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', '', '', '', 'variant summary c p variant cause missense change located protein kinase domain interpro involving alteration conserved nucleotide silico tool predict damaging outcome variant snpsandgo not captured due low reliability index variant absent control chromosome gnomad variant found multiple peutzjeghers syndrome patient addition variant not cited reputable database clinical laboratory taken together variant classified likely pathogenic clinical functional data become available', '', '', '', '', '', '', '', '', '', '', '', 'sequence change replaces valine methionine codon protein p valine residue moderately conserved small physicochemical difference valine methionine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces glycine arginine codon protein p glycine residue highly conserved moderate physicochemical difference glycine arginine variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine glutamic acid codon protein p valine residue highly conserved moderate physicochemical difference valine glutamic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces cysteine phenylalanine codon protein p cysteine residue highly conserved large physicochemical difference cysteine phenylalanine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces aspartic acid glutamic acid codon protein p aspartic acid residue highly conserved small physicochemical difference aspartic acid glutamic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain p amino acid residue determined clinically significant pmid suggests variant disrupt residue likely causative disease summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine isoleucine codon protein p valine residue highly conserved small physicochemical difference valine isoleucine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces methionine valine codon protein p methionine residue highly conserved small physicochemical difference methionine valine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change fall intron gene not directly change encoded amino acid sequence protein affect nucleotide within consensus splice site intron variant present population database exac variant not reported literature individual related disease nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant may disrupt consensus splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces tyrosine cysteine codon protein p tyrosine residue highly conserved large physicochemical difference tyrosine cysteine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine leucine codon protein p valine residue weakly conserved small physicochemical difference valine leucine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result c substitution nucleotide position alanine codon replaced valine amino acid similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces threonine proline codon protein p threonine residue highly conserved small physicochemical difference threonine proline variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces alanine serine codon protein p alanine residue highly conserved moderate physicochemical difference alanine serine variant present population database exac variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'variant gross deletion genomic region encompassing part exon gene includes initiator codon c expected result absent disrupted protein product variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change fall intron gene not directly change encoded amino acid sequence protein affect nucleotide within consensus splice site intron variant not present population database exac frequency variant not reported literature individual related disease nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant may disrupt consensus splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces isoleucine threonine codon protein p isoleucine residue highly conserved moderate physicochemical difference isoleucine threonine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces asparagine glutamic acid codon protein p asparagine residue highly conserved small physicochemical difference asparagine glutamic acid variant not present population database exac frequency variant observed individual affected clinical feature peutz jeghers syndrome pmid invitae clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain variant disrupts p amino acid residue variant disrupt residue observed individual related condition pmid invitae suggesting clinically significant residue result variant disrupt residue likely causative disease summary currently available evidence indicates variant pathogenic additional data needed prove conclusively therefore variant classified likely pathogenic', 'sequence change result frameshift gene p not anticipated result nonsense mediated decay expected disrupt last amino acid protein extend protein additional amino acid variant not present population database frequency information unreliable metric indicate poor data quality position exac database variant not reported literature individual related disease experimental study prediction algorithm not available variant functional significance disrupted amino acid currently unknown summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces alanine valine codon protein p alanine residue moderately conserved small physicochemical difference alanine valine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect acceptor splice site intron gene expected disrupt rna splicing likely result absent disrupted protein product variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pmid observed de novo affected individual invitae donor acceptor splice site variant typically lead loss protein function pmid loss function variant known pathogenic pmid reason variant classified pathogenic', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result substitution nucleotide position methionine codon replaced leucine amino acid highly similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces threonine proline codon protein p threonine residue highly conserved small physicochemical difference threonine proline variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result c g substitution nucleotide position asparagine codon replaced lysine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces aspartic acid asparagine codon protein p aspartic acid residue highly conserved small physicochemical difference aspartic acid asparagine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces cysteine tyrosine codon protein p cysteine residue moderately conserved large physicochemical difference cysteine tyrosine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces alanine threonine codon protein p alanine residue highly conserved small physicochemical difference alanine threonine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class algorithm developed predict effect sequence change rna splicing suggest variant may disrupt consensus splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glutamic acid glutamine codon protein p glutamic acid residue highly conserved small physicochemical difference glutamic acid glutamine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glycine glutamic acid codon protein p glycine residue highly conserved moderate physicochemical difference glycine glutamic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glycine arginine codon protein p glycine residue highly conserved moderate physicochemical difference glycine arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study p amino acid residue determined clinically significant pmid suggests variant disrupt residue likely causative disease summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces lysine arginine codon protein p lysine residue highly conserved small physicochemical difference lysine arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glutamine arginine codon protein p glutamine residue highly conserved small physicochemical difference glutamine arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c c located coding exon gene result c substitution nucleotide position lysine codon replaced glutamine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glutamine arginine codon protein p glutamine residue highly conserved small physicochemical difference glutamine arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'c intronic variant located intron gene result deletion nucleotide within intron gene nucleotide position generally well conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted abolish native splice donor site rna study demonstrated alteration result abnormal splicing set sample tested ambry internal data alteration disrupt canonical splice site expected cause aberrant splicing resulting abnormal protein transcript subject nonsense mediated mrna decay alteration classified likely pathogenic', 'p variant also known c g located coding exon gene result g substitution nucleotide position isoleucine codon replaced valine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position threonine codon replaced alanine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glutamine histidine codon protein p glutamine residue highly conserved small physicochemical difference glutamine histidine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position lysine codon replaced asparagine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', '', 'p variant also known c located coding exon gene result c substitution nucleotide position serine codon replaced leucine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', 'deletion one nucleotide denoted c cdna level p protein level normal sequence base deleted bracket cggg aacc deletion cause frameshift change asparagine threonine codon creates premature stop codon position new reading frame variant predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay c reported association peutz jeghers syndrome papp considered pathogenic', '', '', 'synonymous alteration insufficient evidence classify benign', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', '', '', '', '', '', '', '', '', '', '', '', '', 'synonymous alteration insufficient evidence classify benign', 'sequence change replaces leucine proline codon protein p leucine residue highly conserved moderate physicochemical difference leucine proline variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces aspartic acid glutamic acid codon protein p aspartic acid residue moderately conserved small physicochemical difference aspartic acid glutamic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces lysine glutamic acid codon protein p lysine residue highly conserved small physicochemical difference lysine glutamic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glycine alanine codon protein p glycine residue highly conserved small physicochemical difference glycine alanine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect acceptor splice site intron gene expected disrupt rna splicing likely result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease donor acceptor splice site variant typically lead loss protein function pmid loss function variant known pathogenic pmid summary currently available evidence indicates variant pathogenic additional data needed prove conclusively therefore variant classified likely pathogenic', 'p variant also known c g located coding exon gene result c g substitution nucleotide position glutamine codon replaced glutamic acid amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced tryptophan amino acid highly dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pmid loss function variant known pathogenic pmid reason variant classified pathogenic', 'variant gross deletion genomic region encompassing part exon gene including exon intron boundary c likely creates premature translational stop signal expected result absent disrupted protein product particular variant not reported literature loss function variant known pathogenic pmid reason variant classified pathogenic', '', '', 'sequence change replaces lysine glutamine codon protein p lysine residue highly conserved small physicochemical difference lysine glutamine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine leucine codon protein p valine residue highly conserved small physicochemical difference valine leucine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result g substitution nucleotide position isoleucine codon replaced valine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'sequence change fall intron gene not directly change encoded amino acid sequence protein affect nucleotide within consensus splice site intron variant not present population database exac frequency variant not reported literature individual related disease nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant not likely affect rna splicing prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c c located coding exon gene result g c substitution nucleotide position glutamine codon replaced histidine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pjs segregate disease least one family pmid clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces leucine phenylalanine codon protein p leucine residue highly conserved small physicochemical difference leucine phenylalanine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'p variant also known c g located coding exon gene result c g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces serine proline codon protein p serine residue moderately conserved moderate physicochemical difference serine proline variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class proline amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position amino acid change result serine asparagine codon amino acid highly similar property however change occurs last base pair coding exon make likely effect normal mrna splicing amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis using bdgp esefinder splice site prediction tool alteration predicted weaken efficiency native splice donor site however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position glutamine codon replaced glutamic acid amino acid highly similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant summary c g p result premature termination codon predicted cause truncation encoded protein absence protein due nonsense mediated decay commonly known mechanism disease variant absent control chromosome gnomad c g reported literature multiple individual affected peutz jeghers syndrome schumacher korsse data indicate variant likely associated disease knowledge experimental evidence demonstrating impact protein function reported clinvar submission clinical diagnostic laboratory evaluation cite variant pathogenic based evidence outlined variant classified pathogenic', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces alanine aspartic acid codon protein p alanine residue highly conserved moderate physicochemical difference alanine aspartic acid variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pmid loss function variant known pathogenic pmid reason variant classified pathogenic', 'p variant also known c g located coding exon gene result g substitution nucleotide position glutamine codon replaced arginine amino acid highly similar property amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'synonymous alteration insufficient evidence classify benign', '', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', '', '', 'variant summary c p variant result premature termination codon predicted cause truncated absent protein due nonsense mediated decay commonly known mechanism disease mutation taster predicts damaging outcome variant variant absent control chromosome variant interest not knowledge reported affected individual via publication reputable database clinical diagnostic laboratory evaluated functional impact vivo vitro study taken together variant classified likely pathogenic', '', '', '', '', '', 'synonymous alteration insufficient evidence classify benign', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 'sequence change replaces threonine isoleucine codon protein p threonine residue highly conserved moderate physicochemical difference threonine isoleucine variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', '', 'sequence change replaces glutamic acid alanine codon protein p glutamic acid residue weakly conserved moderate physicochemical difference glutamic acid alanine variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', '', '', '', '', '', '', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', '', '', '', '', '', '', '', '', '', '', '', 'p variant also known c g located coding exon gene result c g substitution nucleotide position leucine codon replaced valine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', '', '', '', '', '', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c c located coding exon gene result g c substitution nucleotide position aspartic acid codon replaced histidine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', 'p variant also known c located coding exon gene result substitution nucleotide position asparagine codon replaced isoleucine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c located coding exon gene result g substitution nucleotide position glutamic acid codon replaced lysine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', 'sequence change replaces serine arginine codon protein p serine residue moderately conserved moderate physicochemical difference serine arginine variant not present population database frequency information unreliable metric indicate poor data quality position exac database variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'sequence change replaces histidine asparagine codon protein p histidine residue weakly conserved small physicochemical difference histidine asparagine variant present population database exac variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'mosaic variant gene not knowledge reported literature substitution creates nonsense variant change tyrosine premature stop codon tac tag predicted cause loss normal protein function protein truncation variant not observed large population cohort lek et al based currently available evidence consider pathogenic', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'p pathogenic mutation also known c g located coding exon gene result c g substitution nucleotide position change amino acid tyrosine stop codon within coding exon alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'c pathogenic mutation located coding exon gene result deletion one nucleotide nucleotide position causing translational frameshift predicted alternate stop codon p alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c g located coding exon gene result g substitution nucleotide position tyrosine codon replaced cysteine amino acid highly dissimilar property amino acid position well conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c c located coding exon gene result g c substitution nucleotide position serine codon replaced threonine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced arginine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position alanine codon replaced valine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'c located untranslated region utr gene andresults deletion thet nucleotide located nucleotidesdownstream last translatedcodon variant not reported population based cohort following database database single nucleotidepolymorphisms dbsnp nhlbiexomesequencing project esp genome project date alteration detected allele frequency greater tested clinical cohort deletion occurs within non coding region gene isnotanticipated disrupt gene function however since supporting evidence limited time clinical significance thec unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position valine codon replaced methionine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'p variant also known c g located coding exon gene result g substitution nucleotide position lysine codon replaced glutamic acid amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c located coding exon gene result g substitution nucleotide position methionine codon replaced isoleucine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result c substitution nucleotide position threonine codon replaced isoleucine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced arginine amino acid dissimilar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result substitution nucleotide position phenylalanine codon replaced tyrosine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'p variant also known c g located coding exon gene result c g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c c located coding exon gene result c substitution nucleotide position leucine codon replaced proline amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'c pathogenic mutation located coding exon gene result deletion one nucleotide nucleotide position causing translational frameshift predicted alternate stop codon p alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'p variant also known c c located coding exon gene result g c substitution nucleotide position aspartic acid codon replaced histidine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position methionine codon replaced valine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position histidine codon replaced glutamine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position serine codon replaced arginine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted benign deleterious polyphen sift silicoanalyses respectively since supporting evidence limited time clinical significance alteration remains unclear', 'c variant span coding exon untranslated region rsquo utr gene variant result duplication nucleotide extends beyond termination codon gene since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced serine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'c c intronic variant result c substitution nucleotide upstream coding exon gene nucleotide position not conserved limited sequence alignment using bdgp esefinder splice site prediction tool alteration not predicted significant effect splice acceptor site however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c c located coding exon gene result c substitution nucleotide position lysine codon replaced threonine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced serine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alterationremains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position serine codon replaced tryptophan amino acid highly dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position not well conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glutamic acid codon replaced lysine amino acid similar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'c c intronic variant result c substitution nucleotide upstream coding exon gene nucleotide position poorly conserved available vertebrate specie bdgp splice prediction software not produce reliable prediction native splice acceptor site esefinder splice prediction software predicts weakening native splice acceptor site efficiency however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result c substitution nucleotide position alanine codon replaced valine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result c substitution nucleotide position glutamine codon replaced lysine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position isoleucine codon replaced methionine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'c g intronic variant result c g substitution nucleotide upstream coding exon gene nucleotide position not well conserved available vertebrate specie bdgp esefinder silico splicing model not produce reliable prediction nearby native splice acceptor site since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c c located coding exon gene result g c substitution nucleotide position lysine codon replaced asparagine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c c located coding exon gene result c substitution nucleotide position lysine codon replaced threonine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position well conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c g located coding exon gene result c g substitution nucleotide position aspartic acid codon replaced glutamic acid amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced histidine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position methionine codon replaced arginine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alterationremains unclear', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c g located coding exon gene result g substitution nucleotide position histidine codon replaced arginine amino acid highly similar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c c located coding exon gene result c substitution nucleotide position methionine codon replaced threonine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position poorly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alterationremains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position methionine codon replaced isoleucine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c c located coding exon gene result g c substitution nucleotide position arginine codon replaced proline amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position valine codon replaced methionine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alterationremains unclear', '', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'c intronic variant result c substitution nucleotide upstream coding exon gene variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort nucleotide position poorly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration not predicted significant effect splice acceptor site however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position threonine codon replaced alanine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c c located coding exon gene result c substitution nucleotide position asparagine codon replaced histidine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position alanine codon replaced threonine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'p variant also known c g located coding exon gene result c g substitution nucleotide position alanine codon replaced glycine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced glutamine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', '', '', 'sequence change replaces proline serine codon protein p proline residue moderately conserved moderate physicochemical difference proline serine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', '', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease however different variant c c giving rise protein effect observed p reported individual affected peutz jeghers syndrome pmid loss function variant known pathogenic pmid reason variant classified pathogenic', 'synonymous alteration insufficient evidence classify benign', 'sequence change replaces isoleucine valine codon protein p isoleucine residue highly conserved small physicochemical difference isoleucine valine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary variant novel missense change uncertain impact protein function classified variant uncertain significance', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'sequence change replaces threonine asparagine codon protein p threonine residue highly conserved small physicochemical difference threonine asparagine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study summary variant novel missense change uncertain impact protein function classified variant uncertain significance', 'synonymous alteration insufficient evidence classify benign', '', 'sequence change replaces alanine proline codon protein p alanine residue highly conserved small physicochemical difference alanine proline variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces valine leucine codon protein p valine residue moderately conserved small physicochemical difference valine leucine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', '', 'synonymous alteration insufficient evidence classify benign', 'sequence change replaces glycine arginine codon protein p glycine residue weakly conserved moderate physicochemical difference glycine arginine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary variant novel missense change uncertain impact protein function classified variant uncertain significance', '', 'sequence change insert nucleotide exon mrna c causing frameshift codon creates premature translational stop signal p expected result absent disrupted protein product particular variant not reported literature loss function variant known pathogenic pmid reason variant classified pathogenic', '', 'synonymous alteration insufficient evidence classify benign', '', '', 'sequence change replaces glycine aspartic acid codon protein p glycine residue moderately conserved moderate physicochemical difference glycine aspartic acid variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class aspartic acid amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result g substitution nucleotide position lysine codon replaced arginine amino acid highly similar property amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant summary c p result conservative amino acid change encoded protein sequence three five silico tool predict benign effect variant protein function variant allele found frequency control chromosome gnomad available data variant occurrence general population insufficient allow conclusion variant significance variant c not reported literature individual affected peutz jeghers syndrome knowledge experimental evidence demonstrating impact protein function reported three clinical diagnostic laboratory submitted clinical significance assessment variant clinvar without evidence independent evaluation laboratory classified variant uncertain significance based evidence outlined variant classified uncertain significance', 'synonymous alteration insufficient evidence classify benign', 'c g intronic variant result c g substitution nucleotide upstream coding exon gene nucleotide position poorly conserved available vertebrate specie neither bdgp esefinder splice site prediction tool able produce reliable prediction nearby native splice acceptor site since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glutamic acid glycine codon protein p glutamic acid residue highly conserved moderate physicochemical difference glutamic acid glycine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces leucine serine codon protein p leucine residue highly conserved large physicochemical difference leucine serine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change affect donor splice site intron gene expected disrupt rna splicing likely result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect sequence change rna splicing suggest variant may disrupt consensus splice site prediction not confirmed published transcriptional study donor acceptor splice site variant typically lead loss protein function pmid loss function variant known pathogenic pmid summary currently available evidence indicates variant pathogenic additional data needed prove conclusively therefore variant classified likely pathogenic', '', '', 'sequence change replaces glutamine histidine codon protein p glutamine residue moderately conserved small physicochemical difference glutamine histidine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary variant uncertain impact function available evidence currently insufficient determine role disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease loss function variant known pathogenic pmid reason variant classified pathogenic', '', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced cysteine amino acid highly dissimilar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'c variant also known p located coding exon gene result deletion gc insertion nucleotide position result substitution serine residue asparagine residue codon amino acid highly similar property amino acid position well conserved available vertebrate specie since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced histidine amino acid highly similar property amino acid position highly conserved available vertebrate specie alteration reported carrier frequency unselected breast cancer patient female control japanese ancestry momozawa et al nat commun addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces proline serine codon protein p proline residue highly conserved moderate physicochemical difference proline serine frequency data variant population database considered unreliable metric indicate insufficient coverage position exac database variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class serine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'sequence change replaces lysine threonine codon protein p lysine residue highly conserved moderate physicochemical difference lysine threonine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study summary variant novel missense change uncertain impact protein function classified variant uncertain significance', '', 'p variant also known c located coding exon gene result c substitution nucleotide position serine codon replaced tyrosine amino acid dissimilar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces arginine glycine codon protein p arginine residue highly conserved moderate physicochemical difference arginine glycine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary variant novel missense change uncertain impact protein function classified variant uncertain significance', 'p variant also known c c located coding exon gene result g c substitution nucleotide position glycine codon replaced arginine amino acid dissimilar property variant reported individual affected ms colorectal cancer age however individual also found heterozygous pathogenic mutation yurgelun mb et al j clin oncol apr amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position asparagine codon replaced lysine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces glycine serine codon protein p glycine residue moderately conserved small physicochemical difference glycine serine variant present population database exac variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class serine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces glycine arginine codon protein p glycine residue moderately conserved moderate physicochemical difference glycine arginine variant not present population database exac frequency variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary variant uncertain impact function available evidence currently insufficient determine role disease therefore classified variant uncertain significance', 'sequence change replaces asparagine serine codon protein p asparagine residue moderately conserved small physicochemical difference asparagine serine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class serine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced alanine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'silico model agreement benign rna study', 'synonymous alteration insufficient evidence classify benign', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', 'pathogenic variation causing aberrant splicing generation truncated protein', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'sequence change replaces leucine proline codon protein p leucine residue highly conserved moderate physicochemical difference leucine proline variant not present population database exac frequency variant observed individual hamartomatous polyp mucocutaneous pigmentation pmid variant also known c c p literature clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', '', '', 'synonymous alteration insufficient evidence classify benign', 'sequence change replaces aspartic acid asparagine codon protein p aspartic acid residue highly conserved small physicochemical difference aspartic acid asparagine frequency data variant population database considered unreliable metric indicate poor data quality position exac database variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'c variant also known p located coding exon gene variant result frame deletion nucleotide cctgctggggga position result deletion four residue codon structural analysis concluded alteration remove key gylcine rich kinase motif thus lead deleterious functional effect protein lahiry p et al nat rev genet jan ambry internal data similar frame deletion gene c reported individual clinical diagnosis peutz jeghers syndrome resta n dig liver dis jul ambry internal data c variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project however position not covered esp amino acid region highly conserved available vertebrate specie based majority available evidence date variant likely pathogenic', 'c c intronic pathogenic mutation result g c substitution one nucleotide coding exon gene though exact alteration not reported literature knowledge different alteration location c designated identified child diagnosed jps history ofhamartomatous polyp perioral pigmentation olschwangs j med genet jun nucleotide position highly conserved available vertebrate specie using thebdgpandesefindersplice site prediction tool alteration predicted abolish native splice donor site however direct experimental evidence unavailable addition clinical data presented literature since alteration disrupt canonical splice donor site typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'c pathogenic mutation located coding exon gene result duplication nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'p variant also known c g located coding exon gene result g substitution nucleotide position leucine codon replaced arginine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glutamic acid codon replaced lysine amino acid similar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'thec g intronic pathogenic mutation result g substitution one nucleotide coding exon gene mutation identified multiple individual peutz jeghers syndrome date resta n hum mutat jul aretz hum mutat dec mehenni h dig dis sci aug additionally rna analysis demonstrated alteration lead use multiple downstream aberrant splice site causing variousframeshift transcript translationally result alternate stop codon hastings ml nat struct mol biol jan based available evidence c g classified pathogenic mutation', 'c intronic pathogenic mutation result g one nucleotide coding exon gene alteration identified multiple affected individual fulfilling clinical diagnostic criterion peutz jeghers syndrome aretz et al hum mutat dec chow e et al clin genet nov hearle n et al clin cancer re may olschwang et al j med genet jun scott rj et al clin genet oct huang z et al bmc gastroenterol nov addition clinical data presented literature alteration disrupt canonical splice site expected cause aberrant splicing resulting abnormal protein transcript subject nonsense mediated mrna decay alteration classified disease causing mutation', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'c pathogenic mutation located coding exon gene result deletion one nucleotide nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'c p pathogenic mutation located coding exon gene result deletion c nucleotide position change amino acid tyrosine stop codon within coding exon since premature stop codon typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'p pathogenic mutation also known c c located coding exon gene result g c substitution nucleotide position arginine codon replaced proline amino acid dissimilar property pathogenic variant confirmed de novo patient meeting clinical criterion pjs well several family meeting clinical criterion pjs variant segregated disease aretz et al hum mutat jiang yl et al bmc med genet ambry internal data addition well characterized mutation affecting codon p also reported patient pjs boudeau j et al hum mutat feb based internal structural analysis variant anticipated result significant decrease structural stability ambry internal data resta n et al cancer re nov based supporting evidence alteration interpreted disease causing mutation', 'c pathogenic mutation located coding exon gene result duplication c nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'p pathogenic mutation also known c located coding exon gene result c substitution nucleotide position change amino acid glutamine stop codon within coding exon x since premature stop codon typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'c g intronic variant result g substitution two nucleotide coding exon gene variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project based nucleotide sequence alignment position highly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted abolish native splice acceptor site however direct evidence unavailable alteration disrupt canonical splice acceptor site typically deleterious nature acmg recommendation standard interpretation reporting sequence variation revision genet med c g variant classified likely pathogenic', 'p variant also known c c located coding exon gene alteration result c substitution nucleotide position leucine codon replaced proline amino acid similar property based protein sequence alignment amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene alteration result g substitution nucleotide position glycine codon replaced serine amino acid similar property based protein sequence alignment amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'c c also known c intronic variant result c substitution two nucleotide coding exon gene variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based nucleotide sequence alignment position highly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted abolish weaken native splice donor site however direct evidence unavailable alteration disrupt canonical splice donor site typically deleterious nature acmg recommendation standard interpretation reporting sequence variation revision genet med c c variant classified likely pathogenic', 'p pathogenic mutation also known c located coding exon gene result c substitution nucleotide position change amino acid glutamine stop codon within coding exon mutation seen multiple patient family peutz jeghers syndrome chow e et al clin genet nov aretz et al hum mutat dec gruber et al cancer re dec zhang lj et al j transl re may addition clinical data presented literature alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'c pathogenic mutation located coding exon gene result duplication g nucleotide position causing translational frameshift predicted alternate stop codon alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'p variant also known c located coding exon gene result substitution nucleotide position isoleucine codon replaced phenylalanine amino acid highly similar property alteration detected one patient meeting diagnostic criterion peutz jeghers syndrome pjs however clinical detail limited borun p et al bmc med genet may variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position poorly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', 'c pathogenic mutation located coding exon gene result deletion nucleotide nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'thec also known p located variant result framedeletion nucleotide ggttctccatccggcaga nucleotide position result deletion acidresidues alteration identified individual clinical diagnosis ofpeutz jeghers syndrome ambry internal data alterationsoccurringin region shown pathogenic wang z hum mutat jul westermanam hum mutat however specific variant not reported literature date variant not reported population based cohort following database database single nucleotidepolymorphisms dbsnp nhlbiexomesequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort deleted amino acid position arewell conserved available vertebrate specie addition deletion predicted highly deleterious proveanin silicoanalysis tool choiy plosone based majority available evidence date variant likely pathogenic', 'c g intronic pathogenic mutation result g substitution two nucleotide upstream coding exon gene c g not reported literature date another alteration location c also referred detected peutz jeghers cohort olschwang et al j med genet jun hearle n et al clin cancer re may addition clinical data presented literature alteration disrupt canonical splice site expected cause aberrant splicing resulting abnormal protein transcript subject nonsense mediated mrna decay alteration classified disease causing mutation', 'c pathogenic mutation located coding exon gene result duplication gaagc nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'c c intronic variant result c substitution two nucleotide upstream coding exon gene nucleotide position highly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted abolish native splice acceptor site however direct evidence unavailable alteration disrupt canonical splice site expected cause aberrant splicing resulting abnormal protein transcript subject nonsense mediated mrna decay alteration classified likely pathogenic', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'co segregation data variant currently unavailable variant not detected conjunction pathogenic mutation date alteration not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project alteration result frame duplication nucleotide position splice prediction software not predict deleterious effect splicing splice prediction software not predict deleterious effect splicing', 'p variant also known c g located coding exon gene alteration result c g substitution nucleotide position leucine codon replaced valine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project based protein sequence alignment amino acid position highly conserved available vertebrate specie addition alteration predicted benign polyphen deleterious sift silico analysis respectively since supporting evidence limited time clinical significance p remains unclear', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'alteration resulting premature truncation e g reading frame shift nonsense', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'c pathogenic mutation located coding exon gene result duplication acct nucleotide position causing translational frameshift predicted alternate stop codon p alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'p pathogenic mutation also known c g located coding exon gene result c g substitution nucleotide position asparagine codon replaced lysine amino acid similar property alteration identified individual suspected peutz jeghers syndrome ambry internal data based internal structural assessment alteration disrupts mg atp binding active site zeqiraj e et al science dec gerlits et al biochemistry may three different alteration codon p p p either reported patient diagnosed peutz jeghers syndrome pjs ylikorkala et al hum mol genet jan patient suspected pjs connolly dc et al j pathol jan amos ci et al j med genet may based supporting evidence alteration interpreted disease causing mutation', 'acmg defined mutation e initiation codon gross deletion', '', 'p variant also known c located coding exon gene result substitution nucleotide position isoleucine codon replaced asparagine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance variantremains unclear', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'p variant also known c c located coding exon gene result g c substitution nucleotide position aspartic acid codon replaced histidine amino acid similar property based internal structural assessment alteration result structural disruption sensitive region protein ambry internal data amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis based majority available evidence date variant likely pathogenic', 'c variant also known p located coding exon gene variant result frame deletion nucleotide position causing removal highly conserved glycine residue codon alteration referred previously identified one individual clinical diagnosis peutz jeghers syndrome pjs found perioral hyperpigmentation well hamartomatous lesion stomach duodenum colon olschwang et al j med genet jun based protein sequence alignment amino acid position highly conserved available vertebrate specie based majority available evidence date variant likely pathogenic', 'alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'p pathogenic mutation also known c g located coding exon gene result c g substitution nucleotide position change amino acid tyrosine stop codon within coding exon mutation reported numerous individual family clinical diagnosis peutz jeghers syndrome hemminki et al nature jan lschwang et al j med genet jun lim w et al gastroenterology jun dai l et al dig dis sci mar addition clinical data presented literature since premature stop codon typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'c variant also known p located coding exon gene variant result frame deletion nucleotide position result deletion four amino acid codon alteration identified individual clinical diagnosis peutz jeghers syndrome resta n dig liver dis jul ambry interal data additionally structural analysis concluded alteration deleteriously affect adjacent essential atp binding loop thus lead deleterious functional effect protein zeqiraj e science dec variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort based protein sequence alignment amino acid highly conserved available vertebrate specie based majority available evidence date variant likely pathogenic', 'p pathogenic mutation also known c g located coding exon gene result c g substitution nucleotide position change amino acid tyrosine stop codon within coding exon mutation reported individual meeting clinical criterion peutz jeghers syndrome lim w br j cancer jul addition another alteration c result nonsense mutation p reported patient hamartomatous polyp stomach small bowl large bowl hearle n clin cancer re may addition clinical data presented literature since premature stop codon typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'nonsense variant gene previously reported least two family peutz jeghers syndrome wang et al wang et al variant predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay based currently available evidence consider pathogenic', 'c variant gene not published pathogenic variant reported benign variant knowledge deletion cause frameshift starting codon proline change amino acid alanine residue creates premature stop codon position new reading frame denoted p variant predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay based currently available evidence consider c pathogenic', 'p variant also known c g located coding exon gene result g substitution nucleotide position lysine codon replaced arginine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant denoted c consists c nucleotide substitution position intron gene variant not knowledge published literature pathogenic benign cysteine c nucleotide altered not conserved across specie silico model inconclusive respect splicing absence rna functional study actual effect variant unknown therefore based currently available information consider c variant uncertain significance', 'sequence change replaces proline leucine codon protein p proline residue highly conserved moderate physicochemical difference proline leucine variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'variant denoted c c cdna level p protein level result change valine alanine gtg gcg variant not knowledge published literature pathogenic benign not observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population since valine alanine share similar property considered conservative amino acid substitution occurs position conserved across specie located c terminal domain hearle silico analysis predict variant probably damaging protein structure function based currently available evidence unclear whether pathogenic benign variant consider variant uncertain significance', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces serine phenylalanine codon protein p serine residue moderately conserved large physicochemical difference serine phenylalanine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary variant uncertain impact function available evidence currently insufficient determine role disease therefore classified variant uncertain significance', 'c variant gene cause frame deletion glutamic acid residue denoted p deletion single residue occurs position conserved across specie located within protein kinase domain uniprot variant not published literature pathogenic benign since frame deletion may may not inhibit proper protein function clinical significance finding remains unclear time consider c variant uncertain significance', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant denoted c consists two nucleotide deletion insertion allele ci position intron gene normal sequence base deleted brace inserted bracket cggg aggg variant not knowledge published literature pathogenic benign multiple splicing model predict c may lead creation cryptic splice donor site absence rna functional study actual effect variant unknown variant not observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population guanine g cytosine c nucleotide altered not conserved based currently available information unclear whether c pathogenic benign consider variant uncertain significance', 'variant denoted c cdna level p protein level normal sequence base deleted brace inserted bracket aaga cgcc frame deletion insertion occurs allele ci result missense change isoleucine lysine atc aag variant not knowledge published literature pathogenic benign neither c alternate nucleotide change observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population since isoleucine lysine share similar property considered conservative amino acid substitution occurs position not conserved not located known functional domain hearle silico analysis inconsistent regarding effect variant may protein structure function based currently available information consider variant uncertain significance', 'p pathogenic mutation also known c located coding exon gene result g substitution nucleotide position change amino acid glutamic acid stop codon within coding exon alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'sequence change replaces serine alanine codon protein p serine residue highly conserved moderate physicochemical difference serine alanine variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', 'variant denoted c g cdna level p protein level result change aspartic acid glycine gac ggc variant not knowledge published literature pathogenic benign however different missense variant involving semi conservative amino acid substitution residue observed several patient family clinical diagnosis peutz jeghers syndrome mehenni de leng dai wang additionally vitro based functional assay showed kinase activity failed induce cell cycle arrest promoter transcription mehenni nezu tiainen not observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population since aspartic acid glycine differ polarity charge size property considered non conservative amino acid substitution occurs position conserved across specie located substrate binding phospho transfer initiation protein kinase domain hearle uniprot silico analysis predict variant probably damaging protein structure function based currently available evidence consider likely pathogenic variant', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced glutamine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant denoted c consists g nucleotide substitution position intron gene multiple silico model predict variant create cryptic donor site intron however absence rna functional study actual effect variant unknown variant not knowledge published literature pathogenic benign c not observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population guanine g nucleotide altered not conserved across specie based currently available information unclear whether c pathogenic benign consider variant uncertain significance', '', 'data supporting benign classification rna study', '', '', 'c variant also known p located coding exon gene variant result frame agg deletion nucleotide position result frame deletion glutamic acid codon amino acid position not well conserved available vertebrate specie addition alteration predicted neutral silico analysis choi et al plo one since supporting evidence limited time clinical significance alteration remains unclear', '', 'variant denoted c cdna level p protein level result change alanine threonine gct act variant not knowledge published literature pathogenic benign not observed approximately individual european african american ancestry nhlbi exome sequencing project indicating not common benign variant population since alanine threonine differ polarity charge size property considered non conservative amino acid substitution occurs position conserved across specie located protein kinase domain region involved binding substrate initiation phospho transfer uniprot hearle silico analysis inconsistent regarding effect variant may protein structure function based currently available information unclear whether pathogenic benign consider variant uncertain significance', 'p variant also known c located coding exon gene result substitution nucleotide position serine codon replaced threonine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position threonine codon replaced alanine amino acid similar property different alteration position p detected individual meeting clinical criterion jps healthy ancestrally diverse individual undergoing wgs liu et al j genet aug bodian dl et al plo one apr amino acid position well conserved available vertebrate specie addition p alteration predicted benign tolerated polyphen sift silico analysis respectively since supporting evidence limited time clinical significance alteration remains unclear', 'variant denoted c cdna level p protein level result change alanine glutamic acid gcg gag variant not knowledge published literature pathogenic benign not observed approximately individual european african american ancestry nhlbi exome sequencing project indicating not common benign variant population since alanine glutamic acid differ polarity charge size property considered non conservative amino acid substitution occurs position not conserved located c terminal region hearle silico analysis predict variant unlikely alter protein structure function based currently available information unclear whether pathogenic benign consider variant uncertain significance', 'variant denoted c consists g nucleotide substitution base pair downstream translational stop codon untranslated region utr gene although variant not knowledge published literature either pathogenic variant benign polymorphism multiple splicing model predict variant may damage natural splice acceptor site exon lead abnormal splicing however absence rna functional study actual effect variant unknown guanine g nucleotide altered conserved mammal based currently available information consider c variant uncertain significance', 'p variant also known c c located coding exon gene result g c substitution nucleotide position valine codon replaced leucine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'c variant gene reported previously association withpeutz jeghers syndrome lim et al ham et al vitro study demonstrated c variant produce truncated protein significantly impairs protein stability ham et al deletion predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay therefore interpret variant pathogenic', 'variant denoted c c cdna level p protein level result change glutamic acid alanine gag gcg variant not knowledge published literature pathogenic benign not observed approximately individual european african american ancestry nhlbi exome sequencing project indicating not common benign variant population since glutamic acid alanine differ polarity charge size property considered non conservative amino acid substitution occurs position conserved throughout vertebrate located within protein kinase domain uniprot silico analysis inconsistent regarding effect variant may protein structure function based currently available information unclear whether pathogenic benign consider variant uncertain significance', 'variant denoted c consists c nucleotide substitution position intron gene multiple silico model predict variant effect nearby natural acceptor site however absence rna functional study actual effect variant unknown variant not knowledge published literature pathogenic benign cytosine c nucleotide altered conserved across specie based currently available information unclear whether c pathogenic benign consider variant uncertain significance', '', '', '', '', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', '', '', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'variant summary c alters non conserved nucleotide located close canonical splice site therefore could affect mrna splicing leading significantly altered protein sequence computational tool predict significant impact normal splicing however prediction yet confirmed functional study variant allele found frequency control chromosome available data variant occurrence general population insufficient allow conclusion variant significance knowledge occurrence c individual affected peutz jeghers syndrome experimental evidence demonstrating impact protein function reported two clinical diagnostic laboratory submitted clinical significance assessment variant clinvar without evidence independent evaluation laboratory classified variant likely benign based evidence outlined variant classified likely benign', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result g substitution nucleotide position glutamic acid codon replaced aspartic acid amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced cysteine amino acid highly dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glycine glutamic acid codon protein p glycine residue moderately conserved moderate physicochemical difference glycine glutamic acid variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced valine amino acid dissimilar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'c variant also known p located coding exon gene variant result frame gcgcgg deletion nucleotide position result deletion two amino acid codon since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c located coding exon gene result c substitution nucleotide position glutamine codon replaced lysine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces aspartic acid asparagine codon protein p aspartic acid residue moderately conserved small physicochemical difference aspartic acid asparagine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'sequence change replaces alanine valine codon protein p alanine residue highly conserved small physicochemical difference alanine valine variant not present population database frequency information unreliable metric indicate poor data quality position exac database variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary variant rare missense change uncertain impact protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'sequence change replaces threonine serine codon protein p threonine residue moderately conserved small physicochemical difference threonine serine variant present population database exac variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class serine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces asparagine threonine codon protein p asparagine residue highly conserved small physicochemical difference asparagine threonine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'sequence change replaces glutamic acid lysine codon protein p glutamic acid residue highly conserved small physicochemical difference glutamic acid lysine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study summary variant novel missense change uncertain impact protein function classified variant uncertain significance', 'sequence change replaces arginine proline codon protein p arginine residue highly conserved moderate physicochemical difference arginine proline variant not present population database rsid frequency information unreliable metric indicate poor data quality position exac database variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study summary variant novel missense change uncertain impact protein function classified variant uncertain significance', 'sequence change replaces glutamic acid alanine codon protein p glutamic acid residue moderately conserved moderate physicochemical difference glutamic acid alanine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class alanine amino acid residue also found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'sequence change replaces glutamine leucine codon protein p glutamine residue highly conserved moderate physicochemical difference glutamine leucine variant not present population database exac frequency not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant rare missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced leucine amino acid dissimilar property alteration reported japanese individual lung cancer onozato r et al cancer sci nov amino acid position well conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change creates premature translational stop signal codon p gene expected result absent disrupted protein product particular variant not reported literature loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change deletes nucleotide exon mrna c causing frameshift codon creates premature translational stop signal p expected result absent disrupted protein product particular variant not reported literature truncating variant known pathogenic pmid reason variant classified pathogenic', 'sequence change replaces tyrosine cysteine codon protein p tyrosine residue moderately conserved large physicochemical difference tyrosine cysteine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen probably damaging align gvgd class summary variant novel missense change uncertain impact protein function classified variant uncertain significance', 'p variant also known c located coding exon gene result c substitution nucleotide position threonine codon replaced methionine amino acid similar property alteration seen unselected patient recurrent metastatic prostate cancer isaacsson velho p et al prostate variant reported pediatric cancer patient underwent whole genome sequencing whole exome sequencing patient diagnosed acute megakaryoblastic leukemia zhang j et al n engl j med dec amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'c variant also known p located coding exon gene result frame deletion ag insertion ct nucleotide position serine codon replaced leucine amino acid dissimilar property amino acid position not well conserved available vertebrate specie since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces alanine threonine codon protein p alanine residue weakly conserved small physicochemical difference alanine threonine variant not present population database frequency information unreliable metric indicate poor data quality position exac database variant not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'p variant also known c c located coding exon gene result c substitution nucleotide position methionine codon replaced threonine amino acid similar property exact variant not reported literature two alteration codon p p previously reported individual peutz jeghers syndrome olschwang et al j med genet jun tchekmedyian et al plo one variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position glutamine codon replaced arginine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant present population database exac not reported literature individual related disease algorithm developed predict effect sequence change rna splicing suggest variant may alter rna splicing prediction not confirmed published transcriptional study summary rare silent change uncertain impact splicing classified variant uncertain significance', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'c intronic variant result g substitution nucleotide coding exon gene nucleotide position poorly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration not predicted significant effect splice donor site however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces aspartic acid asparagine codon protein p aspartic acid residue weakly conserved small physicochemical difference aspartic acid asparagine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'sequence change replaces proline threonine codon protein p proline residue highly conserved small physicochemical difference proline threonine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary variant novel missense change uncertain impact protein function classified variant uncertain significance', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'sequence change replaces lysine glutamine codon protein p lysine residue highly conserved small physicochemical difference lysine glutamine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced arginine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glutamic acid glycine codon protein p glutamic acid residue moderately conserved moderate physicochemical difference glutamic acid glycine variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant novel missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'c pathogenic mutation located coding exon gene result duplication c position causing translational frameshift predicted alternate stop codon mutation observed multiple individual meeting clinical peutz jeghers syndrome pjs criterion schumacher v j med genet may aretz hum mutat dec one paper concludes mutation occurs mutational hotspot reported six unrelated pjs family launonen v hum mutat oct addition clinical data presented literature since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'synonymous alteration insufficient evidence classify benign', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'sequence change result premature translational stop signal gene p not anticipated result nonsense mediated decay expected disrupt last amino acid protein frequency data variant population database considered unreliable metric indicate insufficient coverage position exac database variant not reported literature individual related condition clinvar contains entry variant variation id experimental study prediction algorithm not available variant functional significance affected amino acid currently unknown summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'synonymous alteration insufficient evidence classify benign', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'synonymous alteration insufficient evidence classify benign', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'synonymous alteration insufficient evidence classify benign', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'nonsense variant gene predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay variant not observed approximately individual european african american ancestry nhlbi exome sequencing project indicating not common benign variant population although variant not reported previously knowledge consider pathogenic', '', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', '', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'variant gene previously reported association peutz jeghers syndrome crocker et al variant not observed approximately individual european african american ancestry nhlbi exome sequencing project indicating not common benign variant population variant conservative amino acid substitution not likely impact secondary protein structure residue share similar property substitution occurs position conserved across specie silico analysis predicts variant probably damaging protein structure function missense variant nearby residue reported human gene mutation database association related disorder stenson et al supporting functional importance region protein based currently available evidence strong candidate pathogenic variant however possibility benign cannot excluded', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'synonymous alteration insufficient evidence classify benign', '', '', '', '', '', '', '', '', 'sequence change replaces serine phenylalanine codon protein p serine residue highly conserved large physicochemical difference serine phenylalanine variant not present population database exac frequency variant observed family affected clinical feature peutz jeghers syndrome invitae clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', '', 'p variant also known c located coding exon gene result g substitution nucleotide position glutamic acid codon replaced lysine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'p pathogenic mutation also known c located coding exon gene result c substitution nucleotide position change amino acid glutamine stop codon within coding exon mutation reported multiple individual fulfilling criterion clinical diagnosis peutz jeghers syndrome pjs lim w et al gastroenterology jun hearle n et al clin cancer re may addition clinical data presented literature since premature stop codon typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', '', 'herein report chinese han kindred five pjs patient distributed three generation onset pjs sign three patient rarely early birth identified novel heterozygous mutation c p gene causing short frameshift followed deletion amino acid stk protein mutation co segregated pjs phenotype absent two hundred unrelated ethnicity matched control', '', '', 'c pathogenic mutation located coding exon gene result deletion nucleotide nucleotide position causing translational frameshift predicted alternate stop codon p alteration identified multiple cohort patient clinical diagnosis peutz jegher syndrome resta n et al cancer re nov schumacher v et al j med genet may thakur n et al bmc med genet sep jiang yl et al cancer genet note alteration also designated published literature alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', '', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'c variant located coding exon gene result frame deletion gc insertion ct nucleotide position result substitution glycine residue alanine residue codon amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted neutral silico analysis choi et al plo one since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', '', 'c intronic variant located gene result deletion nucleotide upstream coding exon position using bdgp esefinder splice site prediction tool alteration not predicted significant effect native splice acceptor site however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', '', 'p variant also known c located coding exon gene result c substitution nucleotide position threonine codon replaced isoleucine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c g located coding exon gene result g substitution nucleotide position valine codon replaced glycine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'disease association small case control study data supporting benign classification', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', '', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', '', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', '', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'c variant gene not reported previously pathogenic variant benign variant knowledge deletion cause frameshift starting codonasparagine change amino acid methionine residue creates premature stop codonat position new reading frame denoted p variant predicted tocause loss normal protein function either protein truncation nonsense mediated mrnadecay based currently available evidence consider c pathogenic', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant observed family affected peutz jeghers syndrome pmid clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', 'c intronic variant result g substitution one nucleotide coding exon gene alteration described patient peutz jeghers syndrome colon cancer mehenni h et al gut two alteration nucleotide c c c also described patient peutz jeghers syndrome papp j et al bmc med genet hearle n et al clin cancer re variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual nucleotide position completely conserved sequence alignment using bdgp esefinder splice site prediction tool alteration predicted abolish significantly weaken native splice donor site respectively however direct evidence unavailable alteration disrupt canonical splice donor site typically deleterious nature acmg recommendation standard interpretation reporting sequence variation revision genet med c variant classified likely pathogenic', '', 'synonymous alteration insufficient evidence classify benign', '', '', '', 'p pathogenic mutation also known c g located coding exon gene result g substitution nucleotide position asparagine codon replaced serine amino acid highly similar property alteration identified two first degree relative family suspected peutz jeghers syndrome ambry internal data based internal structural assessment alteration disrupts mg atp binding active site zeqiraj e et al science dec gerlits et al biochemistry may alteration codon p p p p also identified individual either diagnosed suspected peutz jeghers syndrome ambry internal data ylikorkala et al hum mol genet jan connolly dc et al j pathol jan amos ci et al j med genet may based supporting evidence alteration interpreted disease causing mutation', '', 'sequence change replaces glutamic acid aspartic acid codon protein p glutamic acid residue highly conserved small physicochemical difference glutamic acid aspartic acid variant not present population database exac frequency not reported literature individual related disease algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary novel missense change not predicted affect protein function cause disease however evidence insufficient time prove conclusively therefore classified variant uncertain significance', 'synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c g located coding exon gene result g substitution nucleotide position threonine codon replaced alanine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces glycine tryptophan codon protein p glycine residue highly conserved large physicochemical difference glycine tryptophan variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pmid different missense substitution codon p p reported individual related disorder pmid indicates glycine residue important protein function missense change located within functionally conserved kinase domain protein pmid previously reported missense truncating mutation found within domain pmid observation suggest novel missense substitution within domain may affect protein function experiment not done test possibility algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study summary novel missense change reported individual affected related disorder different missense substitution codon reported patient related disorder change located within functionally conserved kinase domain predicted affect protein function however evidence insufficient time prove conclusively classified variant uncertain significance', 'sequence change replaces isoleucine methionine codon protein p isoleucine residue highly conserved small physicochemical difference isoleucine methionine variant not present population database exac frequency however information unreliable metric indicate poor data quality position exac database variant reported individual serrated adenoma lipoma pmid algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen probably damaging align gvgd class summary novel missense change uncertain impact protein function classified variant uncertain significance', '', '', 'sequence change replaces histidine aspartic acid codon protein p histidine residue highly conserved moderate physicochemical difference histidine aspartic acid variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pmid algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study summary rare missense change uncertain impact protein function classified variant uncertain significance', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', '', 'p variant also known c c located coding exon gene result g c substitution nucleotide position arginine codon replaced proline amino acid dissimilar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'variant denoted c cdna level p protein level result change alanine threonine gcc acc variant reported individual suspected peutz jeghers syndrome heinritz not observed large population cohort lek since alanine threonine differ polarity charge size property considered non conservative amino acid substitution not located known functional domain silico analysis includes protein predictor evolutionary conservation support variant not alter protein structure function based currently available evidence unclear whether pathogenic benign variant consider variant uncertain significance', 'sequence change replaces cysteine serine codon protein p cysteine residue moderately conserved moderate physicochemical difference cysteine serine variant not present population database exac frequency not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary variant rare missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'silico model agreement deleterious completely conserved position appropriate specie insufficient conflicting evidence rarity general population database dbsnp esp genome', 'synonymous alteration insufficient evidence classify benign', 'sequence change result premature translational stop signal last coding exon mrna codon p not anticipated result nonsense mediated decay expected create truncated protein variant not present population database exac frequency not reported literature individual related disease functional study regarding potential impact particular truncating variant protein function not reported although vivo study shown c terminal end contains amino acid undergo post translational modification pmid clinical significance finding unclear summary novel truncating variant uncertain impact protein function absence segregation functional data classified variant uncertain significance', 'p variant also known c c located coding exon gene result g c substitution nucleotide position alanine codon replaced proline amino acid highly similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'silico model agreement benign data supporting benign classification synonymous alteration insufficient evidence classify benign', 'synonymous alteration insufficient evidence classify benign', '', 'variant denoted c consists g nucleotide substitution position intron gene multiple silico model predict variant destroy nearby natural donor site possibly cause abnormal gene splicing however absence rna functional study actual effect variant unknown variant not knowledge published literature pathogenic benign c not observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population guanine g nucleotide altered conserved mammal based currently available information unclear whether c pathogenic benign consider variant uncertain significance', '', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced arginine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant denoted c cdna level p protein level normal sequence base deleted brace inserted bracket gcca gc agca frame deletion insertion occurs allele ci result missense change serine asparagine agc aat variant not knowledge published literature pathogenic benign neither c alternate nucleotide change observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population since serine asparagine share similar property considered conservative amino acid substitution occurs position not conserved located c terminal domain hearle silico analysis inconsistent regarding effect variant may protein structure function based currently available information consider variant uncertain significance', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced glutamine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'variant denoted c g g consists g nucleotide substitution position intron gene c g not observed approximately individual european african american ancestry nhlbi exome sequencing project indicating not common benign variant population variant not knowledge published literature pathogenic benign adenine nucleotide altered not conserved across specie intronic variant potential cause incorrect splicing silico splicing model uninformative therefore based currently available information unclear whether c g pathogenic benign variant consider variant uncertain significance', 'variant denoted c g cdna level p protein level result change serine cysteine tct tgt variant not knowledge published literature pathogenic benign not observed approximately individual european african american ancestry nhlbi exome sequencing project suggesting not common benign variant population since serine cysteine differ polarity charge size property considered non conservative amino acid substitution occurs position conserved across specie located domain responsible binding orientation atp hearle silico analysis predict variant probably damaging protein structure function based currently available evidence unclear whether pathogenic benign variant consider variant uncertain significance', 'sequence change replaces glycine aspartic acid codon protein p glycine residue weakly conserved moderate physicochemical difference glycine aspartic acid variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result g substitution nucleotide position lysine codon replaced arginine amino acid highly similar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c g located coding exon gene result c g substitution nucleotide position leucine codon replaced valine amino acid highly similar property amino acid position well conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted create new alternate splice donor site however direct evidence unavailable addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position serine codon replaced leucine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', '', 'duplication one nucleotide denoted c cdna level p protein level normal sequence base duplicated bracket tctt cggc duplication creates nonsense variant change tyrosine premature stop codon variant predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay c observed individual reported peutz jeghers syndrome aretz salloch wang additionally adjacent variant c g c also result premature stop codon residue p reported several individual meeting clinical diagnostic criterion peutz jeghers syndrome wang olschwang lim zhao li therefore consider c pathogenic', 'c variant also known p located coding exon gene variant result frame deletion nucleotide position causing removal well conserved alanine residue codon addition alteration predicted deleterious silico analysis choi et al plo one since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced arginine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance p unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position threonine codon replaced asparagine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position aspartic acid codon replaced glutamic acid amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project however position not covered esp date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p unclear', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'c variant also known p located coding exon gene variant result frame acc deletion nucleotide position result frame deletion proline codon amino acid position well conserved available vertebrate specie since supporting evidence limited time clinical significance alteration remains unclear', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c g located coding exon gene result g substitution nucleotide position glutamine codon replaced arginine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position alanine codon replaced serine amino acid similar property similar variant codon p c reported woman ovarian tumor diagnosed age clinical genetic examination individual revealed slight perioral not buccal hyperpigmentation disseminated facial freckling heinritz w et al onkologie nov amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'thep also known c located result c substitution nucleotide position replaced byglutamicacid amino acid dissimilar property variant not reported population based cohort following database database single nucleotidepolymorphisms dbsnp nhlbiexomesequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c c located coding exon gene result g c substitution nucleotide position glutamic acid codon replaced glutamine amino acid highly similar property amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position histidine codon replaced tyrosine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance p unclear', 'p variant also known c located coding exon gene result substitution nucleotide position phenylalanine codon replaced tyrosine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort based protein sequence alignment thisamino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p unclear', 'thep also known c g located result g substitution nucleotide position replaced byarginine amino acid dissimilar property variant not reported population based cohort following database database single nucleotidepolymorphisms dbsnp nhlbiexomesequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort based protein sequence alignment amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p unclear', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'p variant also known c located coding exon gene result g substitution nucleotide position methionine codon replaced isoleucine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', 'c g intronic pathogenic mutation result g substitution two nucleotide upstream coding exon gene mutation detected multiple individual meeting clinical diagnostic criterion pjs westerman et al hum mut ambry internal data addition clinical data since mutation disrupt canonical splice acceptor site typically deleterious nature variant interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'p variant also known c g located coding exon gene result g substitution nucleotide position aspartic acid codon replaced glycine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'c g intronic variant result c g substitution nucleotide upstream coding exon gene variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort nucleotide position poorly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration not predicted significant effect native splice acceptor site however direct evidence unavailable since supporting evidence limited time clinical significance c g remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position lysine codon replaced glutamic acid amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'thep also known c c located coding exon gene result g c substitution nucleotide position glutamic acid codon replaced glutamine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p remains unclear', '', '', '', '', 'synonymous alteration insufficient evidence classify benign', 'sequence change replaces isoleucine phenylalanine codon protein p isoleucine residue highly conserved small physicochemical difference isoleucine phenylalanine variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change deletes nucleotide exon mrna c causing frameshift codon creates premature translational stop signal p expected result absent disrupted protein product particular variant not reported literature truncating variant known pathogenic pmid reason variant classified pathogenic', 'sequence change fall intron gene not directly change encoded amino acid sequence protein affect nucleotide within consensus splice site intron variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant not likely affect rna splicing prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces aspartic acid glycine codon protein p aspartic acid residue moderately conserved moderate physicochemical difference aspartic acid glycine variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces alanine valine codon protein p alanine residue moderately conserved small physicochemical difference alanine valine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class valine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', '', 'sequence change replaces tryptophan leucine codon protein p tryptophan residue highly conserved small physicochemical difference tryptophan leucine variant not present population database exac frequency not reported literature germline variant three different missense substitution codon p p p one abolished kinase activity vitro pmid reported patient affected peutz jeghers syndrome pmid observation suggest novel missense substitution residue may affect protein function experiment not done test possibility reason variant classified likely pathogenic', 'p variant also known c g located coding exon gene result c g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'sequence change replaces alanine proline codon protein p alanine residue moderately conserved small physicochemical difference alanine proline variant not present population database not reported literature algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study summary novel missense change not predicted affect protein function cause disease however evidence insufficient time prove conclusively classified variant uncertain significance', '', 'sequence change replaces alanine valine codon protein p alanine residue highly conserved small physicochemical difference alanine valine variant present population database rsid variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'synonymous alteration insufficient evidence classify benign', '', 'p variant also known c c located coding exon gene result c substitution nucleotide position leucine codon replaced proline amino acid similar property alteration identified non cancer control individual study looking cancer predisposition mutation patient multiple primary cancer pritchard al et al plo one apr amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c g located coding exon gene result g substitution nucleotide position asparagine codon replaced aspartic acid amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'c variant located coding exon gene result frame deletion ag insertion cc nucleotide position serine residue replaced proline residue proline residue codon amino acid similar property amino acid position not well conserved available vertebrate specie since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign data supporting benign classification', '', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced cysteine amino acid highly dissimilar property amino acid position well conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced leucine amino acid dissimilar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', 'silico model agreement benign strong data supporting benign classification', 'synonymous alteration insufficient evidence classify benign', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant observed individual affected clinical feature peutz jeghers syndrome pmid invitae clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', 'sequence change affect acceptor splice site intron expected disrupt mrna splicing likely result absent disrupted protein product truncating variant known pathogenic particular truncation reported literature pmid reason variant classified likely pathogenic', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', '', '', '', '', '', '', '', 'sequence change replaces glutamic acid lysine codon protein p glutamic acid residue highly conserved small physicochemical difference glutamic acid lysine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change deletes nucleotide exon mrna c causing frameshift codon creates premature translational stop signal p expected result absent disrupted protein product truncating variant known pathogenic particular truncation reported literature individual family affected peutz jeghers syndrome pmid reason variant classified pathogenic', '', 'c variant also known p located coding exon gene result frame deletion cc insertion gt nucleotide position result substitution proline residue valine residue codon amino acid similar property amino acid position highly conserved available vertebrate specie since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position aspartic acid codon replaced asparagine amino acid highly similar property alteration described multiple family meeting clinical diagnostic criterion peutz jeghers syndrome pjs westerman et al hum mutat schumacher v et al j med genet may chow e et al clin genet nov borun p et al bmc med genet may hearle nc et al j med genet apr lim w et al br j cancer jul jiang yl et al cancer genet alteration also reported two unrelated individual juvenile adenomatous hyperplastic polyp ngeow j et al gastroenterology jun alteration also reported individual greece romania turkey undergoing evaluation inherited cancer predisposition tsaousis gn et al bmc cancer jun amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis based majority available evidence date variant likely pathogenic', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'synonymous alteration insufficient evidence classify benign', 'thec pathogenic mutation located coding exon gene result deletion one nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', '', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced alanine amino acid highly similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'c variant located untranslated region rsquo utr gene variant result c substitution base upstream first translated codon nucleotide position well conserved available vertebrate specie since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced alanine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', '', '', 'p variant also known c located coding exon gene result g substitution nucleotide position methionine codon replaced isoleucine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced serine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', '', '', 'p variant also known c g located coding exon gene result g substitution nucleotide position isoleucine codon replaced valine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position glutamic acid codon replaced glycine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c located coding exon gene result g substitution nucleotide position valine codon replaced phenylalanine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual amino acid position highly conserved available vertebrate specie addition alteration predicted benign polyphen deleterious sift silico analysis since supporting evidence limited time clinical significance p remains unclear', '', '', '', 'p variant also known c located coding exon gene result c substitution nucleotide position glutamine codon replaced lysine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance ofp unclear', '', '', 'sequence change replaces arginine tryptophan codon protein p arginine residue weakly conserved moderate physicochemical difference arginine tryptophan variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result c g substitution nucleotide position proline codon replaced arginine amino acid dissimilar property amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces aspartic acid glutamic acid codon protein p aspartic acid residue highly conserved small physicochemical difference aspartic acid glutamic acid variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'thep also known c c located coding exon result g c substitution nucleotide position glutamine codon replaced histidine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project position not covered esp date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position valine codon replaced isoleucine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position highly conserved available vertebrate specie however isoleucine reference amino acid lamprey addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced threonine amino acid highly similar property alteration previously reported least one individual cohort negative individual early onset breast cancer via multiplex panel testing cancer susceptibility gene maxwell kn et al genet med aug amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'silico model agreement benign data supporting benign classification', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'c variant located untranslated region utr gene variant result g substitution nucleotide last coding exon gene variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project based nucleotide sequence alignment nucleotide position not well conserved adenine reference nucleotide elephant since supporting evidence variant limited time clinical significance c remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced tryptophan amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c c located coding exon gene result g c substitution nucleotide position glycine codon replaced alanine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c g located coding exon gene result g substitution nucleotide position asparagine codon replaced serine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', '', '', 'variant summary c p result non conservative amino acid change encoded protein sequence three five silico tool predict benign effect variant protein function variant allele found frequency control chromosome gnomad available data variant occurrence general population insufficient allow conclusion variant significance knowledge occurrence c individual affected peutz jeghers syndrome experimental evidence demonstrating impact protein function reported six clinical diagnostic laboratory submitted clinical significance assessment variant clinvar without evidence independent evaluation laboratory classified variant uncertain significance based evidence outlined variant classified uncertain significance', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', '', '', '', '', 'synonymous alteration insufficient evidence classify benign', 'sequence change replaces arginine tryptophan codon protein p arginine residue moderately conserved moderate physicochemical difference arginine tryptophan frequency data variant population database considered unreliable metric indicate poor data quality position exac database variant reported individual affected gastrointestinal polyposis pmid clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'synonymous alteration insufficient evidence classify benign', 'p variant also known c located coding exon gene result g substitution nucleotide position alanine codon replaced threonine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position well conserved mammal addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', '', 'synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced histidine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result substitution nucleotide position cysteine codon replaced serine amino acid dissimilar property study woman breast cancer unaffected control cameroon uganda variant observed adedokun b et al cancer epidemiol biomarkers prev amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced histidine amino acid highly similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c c located coding exon gene result c substitution nucleotide position glutamic acid codon replaced alanine amino acid dissimilar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces methionine leucine codon protein p methionine residue highly conserved small physicochemical difference methionine leucine variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces arginine serine codon protein p arginine residue highly conserved moderate physicochemical difference arginine serine variant not present population database exac frequency variant observed segregate peutz jeghers syndrome family pmid variant also known literature algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain variant disrupts p amino acid residue variant disrupt residue determined pathogenic pmid suggests residue clinically significant variant disrupt residue likely disease causing reason variant classified pathogenic', 'sequence change replaces arginine glutamine codon protein p arginine residue highly conserved small physicochemical difference arginine glutamine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces alanine threonine codon protein p alanine residue highly conserved small physicochemical difference alanine threonine variant present population database exac variant reported individual affected triple negative breast cancer pmid clinvar contains entry variant variation id experimental study conflicting regard effect missense change activity vitro pmid summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces histidine tyrosine codon protein p histidine residue highly conserved moderate physicochemical difference histidine tyrosine variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function either unavailable not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces aspartic acid asparagine codon protein p aspartic acid residue highly conserved small physicochemical difference aspartic acid asparagine variant not present population database exac frequency variant reported individual affected peutz jeghers syndrome pjs pmid shown segregate pjs single family pmid clinvar contains entry variant variation id experimental study shown although missense change not alter subcellular localization pten binding capacity protein cause loss kinase activity failure phosphorylate pten protein pmid reason variant classified pathogenic', 'sequence change replaces valine methionine codon protein p valine residue highly conserved small physicochemical difference valine methionine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c g located coding exon gene result g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position serine codon replaced tryptophan amino acid highly dissimilar property alteration reported individual clinical feature peutz jeghers syndrome ngeow j et al gastroenterology jun zhang z et al world j gastroenterol apr ambry internal data variant not reported population based cohort genome aggregation database gnomad amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis based majority available evidence date variant likely pathogenic', 'p variant also known c c located coding exon gene result g c substitution nucleotide position glycine codon replaced arginine amino acid dissimilar property variant highly conserved protein kinase domain gene reported multiple individual different population aclinical diagnosis peutz jeghers syndrome barker et al int j gynecol pathol jan fu j genet test mol biomarkers sep another alterationat position p reported mutation patient peutz jeghers syndrome shown lead severely impaired detectable kinase activity lim w gastroenterology jun ylikorkala hum mol genet jan variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis based majority available evidence date variant likely pathogenic', 'p pathogenic mutation also known c located coding exon gene result c substitution nucleotide position change amino acid glutamine stop codon within coding exon alteration previously reported three unrelated individual confirmed suspected peutz jegher syndrome pjs based personal history mucocutaneous hyperpigmentation gastrointestinal polyp without family history pjs related malignancy amos ci et al j med genet may addition clinical data presented literature alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', 'p pathogenic mutation also known c located coding exon gene result c substitution nucleotide position change amino acid cysteine stop codon within coding exon alteration seen peutz jeghers family whose affected member duodenalhamartomatouslesions breast cancer duodenal cancer olschwangs et al j med genet additional clinical information presented literature since premature stop codon typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant found br ov heredic panel', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', 'variant considered likely benign benign based one following criterion conservative change occurs poorly conserved position protein predicted benign multiple silico algorithm population frequency not consistent disease', '', 'synonymous alteration insufficient evidence classify benign', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'synonymous alteration insufficient evidence classify benign', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', '', 'c mutation gene cause frameshift starting codon cysteine change amino acid serine residue creates premature stop codon position new reading frame denoted p mutation predicted cause loss normal protein function either protein truncation nonsense mediated mrna decay although mutation not previously reported knowledge presence consistent diagnosis peutz jeghers syndrome variant found panel', 'c mutation identified gene not published mutation reported benign polymorphism knowledge deletion result loss five highly conserved residue insertion incorrect residue highly conserved functional domain protein similar deletion four residue within highly conserved domain reported family diagnosed peutz jeghers syndrome mehenni et al therefore c interpreted disease causing mutation variant found panel', '', '', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'sequence change replaces threonine methionine codon protein p threonine residue highly conserved moderate physicochemical difference threonine methionine variant not present population database exac frequency variant published literature pmid not reported germline individual related disease clinvar contains entry variant variation id experimental study shown missense change not affect kinase activity protein pmid partially affect efficiency nuclear export ampk activation pmid summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces proline threonine codon protein p proline residue moderately conserved small physicochemical difference proline threonine variant present population database exac variant reported individual affected colorectal cancer pmid clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'p variant also known c located coding exon gene result substitution nucleotide position arginine codon replaced tryptophan amino acid dissimilar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position isoleucine codon replaced methionine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property alteration reported germline patient malignant pleural mesothelioma bueno r et al nat genet apr amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position arginine codon replaced glycine amino acid dissimilar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glutamic acid codon replaced lysine amino acid similar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene alteration result c substitution nucleotide position proline codon replaced serine amino acid similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position highly conserved available vertebrate specie addition alteration predicted possibly damaging deleterious polyphen sift silico analysis respectively since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced glutamine amino acid highly similar property amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces proline leucine codon protein p proline residue moderately conserved moderate physicochemical difference proline leucine variant present population database exac not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift deleterious polyphen benign align gvgd class leucine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study summary variant rare missense change not predicted affect protein function indication cause disease available evidence currently insufficient prove conclusively therefore classified variant uncertain significance', 'sequence change replaces alanine valine codon protein p alanine residue highly conserved small physicochemical difference alanine valine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift tolerated polyphen possibly damaging align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'internal frequency high pathogenic mutation based disease syndrome prevalence penetrance', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'p variant also known c g located coding exon gene result g substitution nucleotide position isoleucine codon replaced methionine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change fall intron gene not directly change encoded amino acid sequence protein affect nucleotide within consensus splice site intron variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id nucleotide substitution within consensus splice site relatively common cause aberrant splicing pmid algorithm developed predict effect sequence change rna splicing suggest variant not likely affect rna splicing prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'sequence change replaces arginine glutamine codon protein p arginine residue moderately conserved small physicochemical difference arginine glutamine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'sequence change replaces arginine cysteine codon protein p arginine residue weakly conserved large physicochemical difference arginine cysteine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'insufficient conflicting evidence classification alteration', 'p variant also known c located coding exon gene result g substitution nucleotide position valine codon replaced methionine amino acid highly similar property alteration detected cohort colorectal cancer patient tested multi gene panel classified variant uncertain significance study author yurgelun mb et al j clin oncol apr amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position serine codon replaced asparagine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c c located coding exon gene result g c substitution nucleotide position glutamic acid codon replaced glutamine amino acid highly similar property amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced histidine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces alanine threonine codon protein p alanine residue weakly conserved small physicochemical difference alanine threonine variant present population database exac variant observed individual affected breast cancer pmid clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class threonine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'p variant also known c located coding exon gene result c substitution nucleotide position alanine codon replaced valine amino acid similar property alteration detected year old female patient msi h colorectal cancer synchronous endometrial cancer undergoing multigene panel testing rohlin et al fam cancer apr amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene variant result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position arginine codon replaced glutamine amino acid highly similar property amino acid position well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result g substitution nucleotide position threonine codon replaced alanine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced tryptophan amino acid dissimilar property alteration reported patient small bowel neuroendocrine tumor however individual also positive pathogenic mutation gene well variant unknown significance fancm gene larouche v et al endocrinol diabetes metab oct amino acid position not well conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'c intronic variant located intron gene result deletion nucleotide within intron gene nucleotide position region generally well conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted weaken efficiency native splice donor site however direct evidence unavailable since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced leucine amino acid similar property amino acid position well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position alanine codon replaced glycine amino acid similar property alteration classified variant uncertain significance identified one time cohort woman breast cancer undergoing multi gene panel testing tung n et al j clin oncol may amino acid position poorly conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign data supporting benign classification strong data supporting benign classification', 'silico model agreement benign strong data supporting benign classification', 'c intronic pathogenic mutation result g substitution one nucleotide coding exon gene nucleotide position highly conserved available vertebrate specie using bdgp esefinder splice site prediction tool alteration predicted abolish native splice donor site however direct evidence unavailable alteration disrupt canonical splice site expected cause aberrant splicing resulting abnormal protein transcript subject nonsense mediated mrna decay alteration classified disease causing mutation', 'p variant also known c c located coding exon gene alteration result g c substitution nucleotide position alanine codon replaced proline amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance p remains unclear', 'c pathogenic mutation located coding exon gene result deletion one nucleotide nucleotide position causing translational frameshift predicted alternate stop codon since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'sequence change replaces valine alanine codon protein p valine residue highly conserved small physicochemical difference valine alanine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function not agree potential impact missense change sift deleterious polyphen benign align gvgd class summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'p variant also known c located coding exon gene alteration result g substitution nucleotide position serine codon replaced isoleucine amino acid dissimilar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project date alteration detected allele frequency approximately greater allele tested clinical cohort includes individual based protein sequence alignment amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance p remains unclear', 'p variant also known c located coding exon gene result g substitution nucleotide position glycine codon replaced serine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'sequence change replaces lysine arginine codon protein p lysine residue moderately conserved small physicochemical difference lysine arginine variant present population database exac variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function output following sift tolerated polyphen benign align gvgd class arginine amino acid residue found multiple mammalian specie suggesting missense change not adversely affect protein function prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'c intronic variant result c substitution nucleotide upstream coding exon gene nucleotide position well conserved available vertebrate specie silico splice site analysis predicts alteration not significant effect splicing since supporting evidence limited time clinical significance alteration remains unclear', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'variant observed part predisposition screen ostensibly healthy population literature search performed gene cdna change amino acid change applicable publication found based search allele frequency data public database allowed determination variant unlikely cause disease therefore variant classified likely benign', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'sequence change replaces serine tryptophan codon protein p serine residue highly conserved large physicochemical difference serine tryptophan frequency data variant population database unreliable metric indicate poor quality position exac database variant observed individual suspected lynch syndrome pmid clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest sequence change likely disruptive prediction not confirmed published functional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'variant summary c p result non conservative amino acid change encoded protein sequence three five silico tool predict benign effect variant protein function variant allele found frequency control chromosome observed variant frequency approximately fold estimated maximal expected allele frequency pathogenic variant causing peutz jeghers syndrome phenotype strongly suggesting variant benign c reported literature individual affected breast cancer report not provide unequivocal conclusion association variant peutz jeghers syndrome knowledge experimental evidence demonstrating impact protein function reported nine clinical diagnostic laboratory submitted clinical significance assessment variant clinvar without evidence independent evaluation laboratory classified variant uncertain significance based evidence outlined variant classified vus possibly benign', 'sequence change replaces alanine threonine codon protein p alanine residue weakly conserved small physicochemical difference alanine threonine variant present population database exac variant reported individual leiden open source variation database pmid clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'p variant also known c g located coding exon gene result g substitution nucleotide position isoleucine codon replaced valine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'sequence change replaces aspartic acid asparagine codon protein p aspartic acid residue highly conserved small physicochemical difference aspartic acid asparagine variant present population database exac variant observed individual affected breast cancer pmid clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely tolerated prediction not confirmed published functional study clinical significance uncertain summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', 'insufficient conflicting evidence classification alteration', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'variant observed part predisposition screen ostensibly healthy population literature search performed gene cdna change amino acid change applicable publication found based search allele frequency data public database allowed determination variant unlikely cause disease therefore variant classified likely benign', '', '', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score variant could not ruled causing disease therefore association disease required investigation literature search performed gene cdna change amino acid change applicable publication found based search variant therefore classified variant unknown significance disease', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', '', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', 'p variant also known c located coding exon gene result g substitution nucleotide position alanine codon replaced threonine amino acid similar property amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'synonymous alteration insufficient evidence classify benign', '', 'sequence change affect codon mrna isilent change meaning not change encoded amino acid sequence protein variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect sequence change rna splicing suggest variant may create strengthen splice site prediction not confirmed published transcriptional study summary available evidence currently insufficient determine role variant disease therefore classified variant uncertain significance', '', 'p variant also known c g located coding exon gene result g substitution nucleotide position glutamic acid codon replaced glycine amino acid similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', 'p variant also known c g located coding exon gene result c g substitution nucleotide position leucine codon replaced valine amino acid highly similar property amino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance alteration remains unclear', '', 'p variant also known c located coding exon gene result c substitution nucleotide position arginine codon replaced tryptophan amino acid dissimilar property alteration detected patient triple negative breast cancer unselected family history breast ovarian cancer couch fj et al j clin oncol feb individual breast cancer referred testing tung n et al cancer jan male patient peutz jeghers syndrome reported variant jiang yl et al bmc med genet alteration reported carrier frequency unselected breast cancer patient female control japanese ancestry momozawa et al nat commun amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'silico model agreement benign synonymous alteration insufficient evidence classify benign', 'alteration classified likely benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', '', '', '', '', '', '', '', 'silico model agreement benign subpopulation frequency support benign classification', 'thep also known c located coding exon result substitution nucleotide position tyrosine codon replaced phenylalanine amino acid highly similar property variant not reported population based cohort following database database single nucleotide polymorphism dbsnp nhlbi exome sequencing project esp genome project esp variant not observed sample allele coverage position date alteration detected allele frequency approximately greater allele tested clinical cohort based proteinsequence alignment thisamino acid position highly conserved available vertebrate specie addition silico prediction alteration inconclusive since supporting evidence limited time clinical significance ofp unclear', 'p variant also known c located coding exon gene result c substitution nucleotide position proline codon replaced glutamine amino acid similar property amino acid position not well conserved available vertebrate specie addition alteration predicted tolerated silico analysis since supporting evidence limited time clinical significance alteration remains unclear', 'sequence change replaces lysine threonine codon protein p lysine residue highly conserved moderate physicochemical difference lysine threonine variant not present population database exac frequency variant not reported literature individual related disease clinvar contains entry variant variation id algorithm developed predict effect missense change protein structure function sift polyphen align gvgd suggest variant likely disruptive prediction not confirmed published functional study clinical significance uncertain summary variant uncertain impact function available evidence currently insufficient determine role disease therefore classified variant uncertain significance', 'data supporting benign classification rna study', '', '', '', '', '', 'co occurence mutation another gene clearly explains proband phenotype silico model agreement benign', '', 'alteration classified benign based combination following population frequency intact protein function lack segregation disease co occurrence rna analysis silico model amino acid conservation lack disease association case control study mechanism disease impacted region inconsistent known cause pathogenicity', '', 'variant observed icsl laboratory part predisposition screen ostensibly healthy population not previously curated icsl reported human gene mutation database hgmd prior june therefore candidate classification automated scoring system utilizing variant allele frequency disease prevalence penetrance estimate inheritance mode automated score calculated ass variant frequent cause disease based score internal cut value variant classified benign not subjected curation score variant resulted classification benign disease', '', '', '', '', '', '', '', '', '', 'sequence change creates premature translational stop signal p gene expected result absent disrupted protein product variant not present population database exac frequency variant not reported literature individual related condition clinvar contains entry variant variation id loss function variant known pathogenic pmid reason variant classified pathogenic', '', '', '', 'c pathogenic mutation located coding exon gene result deletion one nucleotide position causing translational frameshift predicted alternate stop codon mutation reported several family diagnosed peutz jeghers syndrome nakagawa et al hum genet aug lim et al gastroenterology jun amos et al j med genet may de leng et al clin genet dec addition clinical data presented literature since frameshifts typically deleterious nature alteration interpreted disease causing mutation acmg recommendation standard interpretation reporting sequence variation revision genet med', 'silico model agreement deleterious completely conserved position appropriate specie detected individual satisfying established diagnostic critera classic disease without clear mutation well characterized mutation position', '', '', 'p variant also known c c located coding exon gene result c substitution nucleotide position leucine codon replaced proline amino acid similar property alteration seen multiple unrelated individual meeting diagnostic criterion peutz jeghers syndrome hemminki et al nature jan hearle n et al clin cancer re may ambry internal data variant previously reported snpdatabase date alteration detected allele frequency approximately greater allele tested clinical cohort amino acid position highly conserved available vertebrate specie addition alteration predicted deleterious silico analysis based majority available evidence date variant likely pathogenic', 'p pathogenic mutation also known c located coding exon gene result substitution nucleotide position change amino acid lysine stop codon within coding exon mutation reported several unrelated individual diagnosed peutz jeghers syndrome hemminki et al nature jan lim w et al gastroenterology jun mehenni h et al dig dis sci aug jiang et al bmc med genet addition clinical data presented literature alteration expected result loss function premature protein truncation nonsense mediated mrna decay alteration interpreted disease causing mutation', '', '', '', '']\n"
     ]
    }
   ],
   "source": [
    "corpus = []\n",
    "for i in range(len(dataset.Description)):\n",
    "    review = re.sub('[^a-zA-Z]' ,' ', dataset['Description'][i])\n",
    "    review =review.lower()\n",
    "    review = review.split()\n",
    "    all_stopwords = stopwords.words('english')\n",
    "    all_stopwords.remove('not')\n",
    "    review = [lemmatizer.lemmatize(word) for word in review  if not word in set(all_stopwords)]\n",
    "    review = ' '.join(review)\n",
    "    corpus.append(review)\n",
    "print(corpus)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "cv = TfidfVectorizer()\n",
    "av = CountVectorizer()\n",
    "X_des = cv.fit_transform(corpus).toarray()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       ...,\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.]])"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_des"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "970"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(X_des[0])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.4711947 , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.20128092, 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.4609082 , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.36339164, 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.19302843, 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.26460757, 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.32935901, 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.21251757, 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.36339164, 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ,\n",
       "       0.        , 0.        , 0.        , 0.        , 0.        ])"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_des[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[1., 1., 0., ..., 1., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 1., 0., 0.],\n",
       "       ...,\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.]])"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Y"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       ...,\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.],\n",
       "       [0., 0., 0., ..., 0., 0., 0.]])"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_des"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "##import tensorflow as tf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "##ann = tf.keras.models.Sequential()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "##ann.add(tf.keras.layers.Dense(units = 10 , activation='relu'))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "##ann.add(tf.keras.layers.Dense(units = 50 , activation='relu'))\n",
    "\n",
    "##ann.add(tf.keras.layers.Dense(units =  , activation='relu'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "##ann.add(tf.keras.layers.Dense(units = 27 , activation='sigmoid'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "##ann.compile(optimizer = 'adam' , loss = 'categorical_crossentropy', metrics = ['accuracy'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "##ann.fit(X_des , Y , batch_size = 10 , epochs = 10)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax (<ipython-input-33-408eb292872b>, line 1)",
     "output_type": "error",
     "traceback": [
      "\u001b[1;36m  File \u001b[1;32m\"<ipython-input-33-408eb292872b>\"\u001b[1;36m, line \u001b[1;32m1\u001b[0m\n\u001b[1;33m    from sklearn import\u001b[0m\n\u001b[1;37m                       ^\u001b[0m\n\u001b[1;31mSyntaxError\u001b[0m\u001b[1;31m:\u001b[0m invalid syntax\n"
     ]
    }
   ],
   "source": [
    "from sklearn import"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
